<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1 Introduction | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis</title>
  <meta name="description" content="1 Introduction | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  <meta name="generator" content="bookdown 0.23 and GitBook 2.6.7" />

  <meta property="og:title" content="1 Introduction | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="Huang-AI4Medicine-Lab/survey-of-DL-for-scRNA-seq-analysis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1 Introduction | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  
  
  

<meta name="author" content="Mario Flores1§, Zhentao Liu1, Tinghe Zhang1, Md Musaddaqui Hasib1, Yu-Chiao Chiu2, Zhenqing Ye2,3, Karla Paniagua1, Sumin Jo1, Jianqiu Zhang1, Shou-Jiang Gao4,6, Yufang Jin1, Yidong Chen2,3§, and Yufei Huang5,6§" />


<meta name="date" content="2021-09-13" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="about-this-book.html"/>
<link rel="next" href="ch-2.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Survey of Deep Learning for scRNA-seq Analysis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About the authors</a></li>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html"><i class="fa fa-check"></i>About this book</a>
<ul>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html#key-points"><i class="fa fa-check"></i>Key Points</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="ch-1.html"><a href="ch-1.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="ch-2.html"><a href="ch-2.html"><i class="fa fa-check"></i><b>2</b> Overview of scRNA-seq processing pipeline</a></li>
<li class="chapter" data-level="3" data-path="ch-3.html"><a href="ch-3.html"><i class="fa fa-check"></i><b>3</b> Overview of common deep learning models for scRNA-seq analysis</a>
<ul>
<li class="chapter" data-level="3.1" data-path="ch-3.html"><a href="ch-3.html#ch-3-1"><i class="fa fa-check"></i><b>3.1</b> Variational Autoencoder (VAEs) for scRNA-Seq data</a></li>
<li class="chapter" data-level="3.2" data-path="ch-3.html"><a href="ch-3.html#ch-3-2"><i class="fa fa-check"></i><b>3.2</b> Autoencoders (AEs) for scRNA-seq data</a></li>
<li class="chapter" data-level="3.3" data-path="ch-3.html"><a href="ch-3.html#ch-3-3"><i class="fa fa-check"></i><b>3.3</b> Generative adversarial networks (GANs) for scRNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-4.html"><a href="ch-4.html"><i class="fa fa-check"></i><b>4</b> Overview of datasets and evaluation metrics</a>
<ul>
<li class="chapter" data-level="4.1" data-path="ch-4.html"><a href="ch-4.html#ch-4-1"><i class="fa fa-check"></i><b>4.1</b> Evaluation methods</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-1"><i class="fa fa-check"></i><b>4.1.1</b> Imputation</a></li>
<li class="chapter" data-level="4.1.2" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-2"><i class="fa fa-check"></i><b>4.1.2</b> Batch effect correction</a></li>
<li class="chapter" data-level="4.1.3" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-3"><i class="fa fa-check"></i><b>4.1.3</b> Clustering</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-5.html"><a href="ch-5.html"><i class="fa fa-check"></i><b>5</b> Survey of deep learning models for scRNA-Seq analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-1"><i class="fa fa-check"></i><b>5.1</b> Imputation</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-1"><i class="fa fa-check"></i><b>5.1.1</b> DCA:deep count autoencoder</a></li>
<li class="chapter" data-level="5.1.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-2"><i class="fa fa-check"></i><b>5.1.2</b> SAVER-X: single-cell analysis via expression recovery harnessing external data</a></li>
<li class="chapter" data-level="5.1.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-3"><i class="fa fa-check"></i><b>5.1.3</b> DeepImpute (Deep neural network Imputation)</a></li>
<li class="chapter" data-level="5.1.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-4"><i class="fa fa-check"></i><b>5.1.4</b> LATE: Learning with AuToEncoder</a></li>
<li class="chapter" data-level="5.1.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-5"><i class="fa fa-check"></i><b>5.1.5</b> scGMAI</a></li>
<li class="chapter" data-level="5.1.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-6"><i class="fa fa-check"></i><b>5.1.6</b> scIGANs</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-2"><i class="fa fa-check"></i><b>5.2</b> Batch effect correction</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-1"><i class="fa fa-check"></i><b>5.2.1</b> BERMUDA: Batch Effect ReMoval Using Deep Autoencoders</a></li>
<li class="chapter" data-level="5.2.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-2"><i class="fa fa-check"></i><b>5.2.2</b> DESC: batch correction based on clustering</a></li>
<li class="chapter" data-level="5.2.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-3"><i class="fa fa-check"></i><b>5.2.3</b> iMAP: Integration of Multiple single-cell datasets by Adversarial Paired-style transfer networks</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-3"><i class="fa fa-check"></i><b>5.3</b> Dimension reduction, latent representation, clustering, and data augmentation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-1"><i class="fa fa-check"></i><b>5.3.1</b> Dimension reduction by AEs with gene-interaction constrained architecture</a></li>
<li class="chapter" data-level="5.3.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-2"><i class="fa fa-check"></i><b>5.3.2</b> Dhaka: a VAE-based dimension reduction model</a></li>
<li class="chapter" data-level="5.3.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-3"><i class="fa fa-check"></i><b>5.3.3</b> cvis: a VAE for capturing low-dimensional structures</a></li>
<li class="chapter" data-level="5.3.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-4"><i class="fa fa-check"></i><b>5.3.4</b> scVAE: VAE for single-cell gene expression data</a></li>
<li class="chapter" data-level="5.3.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-5"><i class="fa fa-check"></i><b>5.3.5</b> VASC: VAE for scRNA-seq</a></li>
<li class="chapter" data-level="5.3.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-6"><i class="fa fa-check"></i><b>5.3.6</b> scDeepCluster</a></li>
<li class="chapter" data-level="5.3.7" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-7"><i class="fa fa-check"></i><b>5.3.7</b> cscGAN: Conditional single-cell generative adversarial neural networks</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-4"><i class="fa fa-check"></i><b>5.4</b> Multi-functional models</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-1"><i class="fa fa-check"></i><b>5.4.1</b> scVI: single-cell variational inference</a></li>
<li class="chapter" data-level="5.4.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-2"><i class="fa fa-check"></i><b>5.4.2</b> LDVAE: linearly decoded variational autoencoder</a></li>
<li class="chapter" data-level="5.4.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-3"><i class="fa fa-check"></i><b>5.4.3</b> SAUCIE</a></li>
<li class="chapter" data-level="5.4.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-4"><i class="fa fa-check"></i><b>5.4.4</b> </a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-5"><i class="fa fa-check"></i><b>5.5</b> Doublet Classification</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-5-1"><i class="fa fa-check"></i><b>5.5.1</b> Solo</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-6"><i class="fa fa-check"></i><b>5.6</b> Automated cell type identification</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-1"><i class="fa fa-check"></i><b>5.6.1</b> DigitalDLSorter</a></li>
<li class="chapter" data-level="5.6.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-2"><i class="fa fa-check"></i><b>5.6.2</b> scCapsNet</a></li>
<li class="chapter" data-level="5.6.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-3"><i class="fa fa-check"></i><b>5.6.3</b> netAE: network-enhanced autoencoder</a></li>
<li class="chapter" data-level="5.6.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-4"><i class="fa fa-check"></i><b>5.6.4</b> scDGN - supervised adversarial alignment of single-cell RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="ch-5.html"><a href="ch-5.html#ch-5-7"><i class="fa fa-check"></i><b>5.7</b> Biological function prediction</a>
<ul>
<li class="chapter" data-level="5.7.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-7-1"><i class="fa fa-check"></i><b>5.7.1</b> CNNC: convolutional neural network for coexpression</a></li>
<li class="chapter" data-level="5.7.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-7-2"><i class="fa fa-check"></i><b>5.7.2</b> scGen, a generative model to predict perturbation response of single cells across cell types</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="ch-6.html"><a href="ch-6.html"><i class="fa fa-check"></i><b>6</b> Conclusion and Discussions</a></li>
<li class="chapter" data-level="7" data-path="ch-7.html"><a href="ch-7.html"><i class="fa fa-check"></i><b>7</b> Tables</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch-1" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Single cell sequencing technology has been a rapidly developing area to study genomics, transcriptomics, proteomics, metabolomics, and cellular interactions at the single cell level for cell-type identification, tissue composition and reprogramming <span class="citation">(<a href="#ref-RN87" role="doc-biblioref">Lahnemann et al. 2020</a>; <a href="#ref-RN98" role="doc-biblioref">Vitak et al. 2017</a>)</span> <!-- [1,2].  -->. Specifically, sequencing of the transcriptome of single cells, or single-cell RNA-sequencing (scRNA-seq), has become the dominant technology in many frontier research areas such as disease progression and drug discovery <span class="citation">(<a href="#ref-RN191" role="doc-biblioref">Wolock, Lopez, and Klein 2019</a>; <a href="#ref-RN78" role="doc-biblioref">Bost et al. 2020</a>)</span> <!-- [3, 4] -->. One particular area where scRNA-seq has made a tangible impact is cancer, where scRNA-seq is becoming a powerful tool for understanding invasion, intratumor heterogeneity, metastasis, epigenetic alterations, detecting rare cancer stem cells, and therapeutic response (refs). Currently, scRNA-seq is applied to develop personalized therapeutic strategies that are potentially useful in cancer diagnosis, therapy resistance during cancer progression, and the survival of patients <span class="citation">(<a href="#ref-RN85" role="doc-biblioref">Kinker et al. 2020</a>; <a href="#ref-RN59" role="doc-biblioref">Navin 2015</a>)</span><!-- [5, 6] -->. The scRNA-seq has also been adopted to combat COVID-19 to elucidate how the innate and adaptive host immune system miscommunicates resulting in worsening the immunopathology produced during this viral infection <span class="citation">(<a href="#ref-RN91" role="doc-biblioref">Mannarapu, Dariya, and Bandapalli 2021</a>; <a href="#ref-RN99" role="doc-biblioref">Wauters et al. 2021</a>)</span><!-- [7, 8] -->.</p>
<p>These studies have led to a massive amount of scRNA-seq data deposited to public databases such as 10X Single-cell gene expression dataset, Human Cell Atlas, and Mouse Cell Atlas. Expressions of millions of cells from 18 species have been collected and deposited, waiting for further analysis. On the other hand, due to biological and technical factors, scRNA-seq data presents several analytical challenges related to its complex characteristics like missing expression values, high technical and biological variance, noise and sparse gene coverage, and elusive cell identities <span class="citation">(<a href="#ref-RN87" role="doc-biblioref">Lahnemann et al. 2020</a>)</span> <!--[1]-->. These characteristics make it difficult to directly apply commonly used bulk RNA-seq data analysis techniques and have called for novel statistical approaches for scRNA-seq data cleaning and computational algorithms for data analysis and interpretation. To this end, specialized scRNA-seq analysis pipelines such as Seurat <span class="citation">(<a href="#ref-RN96" role="doc-biblioref">Stuart et al. 2019</a>)</span><!--[9]--> and Scanpy <span class="citation">(<a href="#ref-RN100" role="doc-biblioref">Wolf, Angerer, and Theis 2018</a>)</span><!--[10]-->. along with a large collection of task-specific tools, have been developed to address the intricate technical and biological complexity of scRNA-seq data.</p>
<p>Recently, deep learning has demonstrated its significant advantages in natural language processing and speech and facial recognition with massive data <span class="citation">(<a href="#ref-RN95" role="doc-biblioref">Srinivasan et al. 2020</a>; <a href="#ref-RN76" role="doc-biblioref">Amodio et al. 2019</a>; <a href="#ref-RN88" role="doc-biblioref">Lopez et al. 2018</a>)</span><!--[11-13]-->. Such advantages have initiated the application of DL in scRNA-seq data analysis as a competitive alternative to conventional machine learning approaches for uncovering cell clustering [14, 15], cell type identification [14, 16], gene imputation [17-19], and batch correction [20] in scRNA-seq analysis. Compared to conventional machine learning (ML) approaches, DL is more powerful in capturing complex features of high-dimensional scRNA-seq data. It is also more versatile , where a single model can be trained to address multiple tasks or adapted and transferred to different tasks. Moreover, the DL training scales more favorably with the number of cells in scRNA-seq data size, making it particularly attractive for handling the ever-increasing volume of single cell data. Indeed, the growing body of DL-based tools has demonstrated DL’s exciting potential as a learning paradigm to significantly advance the tools we use to interrogate scRNA-seq data.</p>
<p>In this paper, we present a comprehensive review of the recent advances of DL methods for solving the present challenges in scRNA-seq data analysis (Table<a href="ch-7.html#fig:Table1">7.1</a>) from the quality control, normalization/batch effect reduction, dimension reduction, visualization, feature selection, and data interpretation by surveying deep learning papers published up to April 2021. In order to maintain high quality for this review, we choose not to include any (bio)archival papers, although a proportion of these manuscripts contain important new findings that would be published after completing their peer-reviewed process. Previous efforts to review the recent advances in machine learning methods focused on efficient integration of single cell data [21, 22]. A recent review of DL applications on single cell data has summarized 21 DL algorithms that might be deployed in single cell studies [23]. It also evaluated the clustering and data correction effect of these DL algorithms using 11 datasets.</p>
<p>In this review, we focus more on the DL algorithms with a much detailed explanation and comparison. Further, to better understand the relationship of each surveyed DL model with the overall scRNA-seq analysis pipeline, we organize the surveys according to the challenge they address and discuss these DL models following the analysis pipeline. A unified mathematical description of the surveyed DL models is presented and the specific model features are discussed when reviewing each method. This will also shed light on the modeling connections among the surveyed DL methods and the recognization of the uniqueness of each model. Besides the models, we also summarize the evaluation matrics of these DL algorithms and compare the tools that integrate these DL algorithms. Access to these DL algorithms with the original research results, available datasets used by these methods are also listed to demonstrate the advantages and utility of the DL algorithms. We envision that this survey will serve as an important information portal for learning the application of DL for scRNA-seq analysis and inspire innovative use of DL to address a broader range of new challenges in emerging multi-omics and spatial single-cell sequencing.</p>
<!-- 1  Lahnemann D, Koster J, Szczurek E, McCarthy DJ, Hicks SC, Robinson MD, Vallejos CA, Campbell KR, Beerenwinkel N, Mahfouz A et al: Eleven grand challenges in single-cell data science. Genome Biol 2020, 21(1):31. -->
<!-- 2  Vitak SA, Torkenczy KA, Rosenkrantz JL, Fields AJ, Christiansen L, Wong MH, Carbone L, Steemers FJ, Adey A: Sequencing thousands of single-cell genomes with combinatorial indexing. Nat Methods 2017, 14(3):302-308. -->
<!-- 3  Wu H, Wang C, Wu S: Single-Cell Sequencing for Drug Discovery and Drug Development. Curr Top Med Chem 2017, 17(15):1769-1777. -->
<!-- 4  Bost P, Giladi A, Liu Y, Bendjelal Y, Xu G, David E, Blecher-Gonen R, Cohen M, Medaglia C, Li H et al: Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients. Cell 2020, 181(7):1475-1488 e1412. -->
<!-- 5  Kinker GS, Greenwald AC, Tal R, Orlova Z, Cuoco MS, McFarland JM, Warren A, Rodman C, Roth JA, Bender SA et al: Pan-cancer single-cell RNA-seq identifies recurring programs of cellular heterogeneity. Nat Genet 2020, 52(11):1208-1218. -->
<!-- 6  Navin NE: The first five years of single-cell cancer genomics and beyond. Genome Res 2015, 25(10):1499-1507. -->
<!-- 7  Mannarapu M, Dariya B, Bandapalli OR: Application of single-cell sequencing technologies in pancreatic cancer. Mol Cell Biochem 2021, 476(6):2429-2437. -->
<!-- 8  Wauters E, Van Mol P, Garg AD, Jansen S, Van Herck Y, Vanderbeke L, Bassez A, Boeckx B, Malengier-Devlies B, Timmerman A et al: Discriminating mild from critical COVID-19 by innate and adaptive immune single-cell profiling of bronchoalveolar lavages. Cell Res 2021, 31(3):272-290. -->
<!-- 9  Stuart T, Butler A, Hoffman P, Hafemeister C, Papalexi E, Mauck WM, 3rd, Hao Y, Stoeckius M, Smibert P, Satija R: Comprehensive Integration of Single-Cell Data. Cell 2019, 177(7):1888-1902 e1821. -->
<!-- 10 Wolf FA, Angerer P, Theis FJ: SCANPY: large-scale single-cell gene expression data analysis. Genome Biol 2018, 19(1):15. -->
<!-- 11 Srinivasan S, Leshchyk A, Johnson NT, Korkin D: A hybrid deep clustering approach for robust cell type profiling using single-cell RNA-seq data. RNA 2020, 26(10):1303-1319. -->
<!-- 12 Amodio M, Van Dijk D, Srinivasan K, Chen WS, Mohsen H, Moon KR, Campbell A, Zhao Y, Wang X, Venkataswamy M: Exploring single-cell data with deep multitasking neural networks. Nature methods 2019:1-7. -->
<!-- 13 Lopez R, Regier J, Cole MB, Jordan MI, Yosef N: Deep generative modeling for single-cell transcriptomics. Nat Methods 2018, 15(12):1053-1058. -->
<!-- 14 Amodio M, van Dijk D, Srinivasan K, Chen WS, Mohsen H, Moon KR, Campbell A, Zhao Y, Wang X, Venkataswamy M et al: Exploring single-cell data with deep multitasking neural networks. Nat Methods 2019, 16(11):1139-1145. -->
<!-- 15 Eraslan G, Simon LM, Mircea M, Mueller NS, Theis FJ: Single-cell RNA-seq denoising using a deep count autoencoder. Nat Commun 2019, 10(1):390. -->
<!-- 16 Xu Y, Zhang Z, You L, Liu J, Fan Z, Zhou X: scIGANs: single-cell RNA-seq imputation using generative adversarial networks. Nucleic Acids Res 2020, 48(15):e85. -->
<!-- 17 Arisdakessian C, Poirion O, Yunits B, Zhu X, Garmire LX: DeepImpute: an accurate, fast, and scalable deep neural network method to impute single-cell RNA-seq data. Genome Biol 2019, 20(1):211. -->
<!-- 18 Tran HTN, Ang KS, Chevrier M, Zhang X, Lee NYS, Goh M, Chen J: A benchmark of batch-effect correction methods for single-cell RNA sequencing data. Genome Biol 2020, 21(1):12. -->
<!-- 19 Petegrosso R, Li Z, Kuang R: Machine learning and statistical methods for clustering single-cell RNA-sequencing data. Brief Bioinform 2020, 21(4):1209-1223. -->
<!-- 20 Abdelaal T, Michielsen L, Cats D, Hoogduin D, Mei H, Reinders MJT, Mahfouz A: A comparison of automatic cell identification methods for single-cell RNA sequencing data. Genome Biol 2019, 20(1):194. -->
<!-- 21 Picelli S, Bjorklund AK, Faridani OR, Sagasser S, Winberg G, Sandberg R: Smart-seq2 for sensitive full-length transcriptome profiling in single cells. Nat Methods 2013, 10(11):1096-1098. -->
<!-- 22 Macosko EZ, Basu A, Satija R, Nemesh J, Shekhar K, Goldman M, Tirosh I, Bialas AR, Kamitaki N, Martersteck EM et al: Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. Cell 2015, 161(5):1202-1214. -->
<!-- 23 Chen G, Ning B, Shi T: Single-Cell RNA-Seq Technologies and Related Computational Data Analysis. Front Genet 2019, 10:317. -->
<!-- 24 Eisenstein M: Single-cell RNA-seq analysis software providers scramble to offer solutions. Nat Biotechnol 2020, 38(3):254-257. -->
<!-- 25 Vallejos CA, Marioni JC, Richardson S: BASiCS: Bayesian Analysis of Single-Cell Sequencing Data. PLoS Comput Biol 2015, 11(6):e1004333. -->
<!-- 26 Lun AT, Bach K, Marioni JC: Pooling across cells to normalize single-cell RNA sequencing data with many zero counts. Genome Biol 2016, 17:75. -->
<!-- 27 Hafemeister C, Satija R: Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. Genome Biol 2019, 20(1):296. -->
<!-- 28 Haghverdi L, Lun ATL, Morgan MD, Marioni JC: Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors. Nat Biotechnol 2018, 36(5):421-427. -->
<!-- 29 Butler A, Hoffman P, Smibert P, Papalexi E, Satija R: Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat Biotechnol 2018, 36(5):411-420. -->
<!-- 30 Korsunsky I, Millard N, Fan J, Slowikowski K, Zhang F, Wei K, Baglaenko Y, Brenner M, Loh PR, Raychaudhuri S: Fast, sensitive and accurate integration of single-cell data with Harmony. Nat Methods 2019, 16(12):1289-1296. -->
<!-- 31 Peng T, Zhu Q, Yin P, Tan K: SCRABBLE: single-cell RNA-seq imputation constrained by bulk RNA-seq data. Genome Biol 2019, 20(1):88. -->
<!-- 32 Huang M, Wang J, Torre E, Dueck H, Shaffer S, Bonasio R, Murray JI, Raj A, Li M, Zhang NR: SAVER: gene expression recovery for single-cell RNA sequencing. Nat Methods 2018, 15(7):539-542. -->
<!-- 33 Li WV, Li JJ: An accurate and robust imputation method scImpute for single-cell RNA-seq data. Nat Commun 2018, 9(1):997. -->
<!-- 34 Roweis ST, Saul LK: Nonlinear dimensionality reduction by locally linear embedding. Science 2000, 290(5500):2323-2326. -->
<!-- 35 Welch JD, Hartemink AJ, Prins JF: SLICER: inferring branched, nonlinear cellular trajectories from single cell RNA-seq data. Genome Biol 2016, 17(1):106. -->
<!-- 36 Linderman GC, Rachh M, Hoskins JG, Steinerberger S, Kluger Y: Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data. Nat Methods 2019, 16(3):243-245. -->
<!-- 37 Becht E, McInnes L, Healy J, Dutertre CA, Kwok IWH, Ng LG, Ginhoux F, Newell EW: Dimensionality reduction for visualizing single-cell data using UMAP. Nat Biotechnol 2018. -->
<!-- 38 Subelj L, Bajec M: Unfolding communities in large complex networks: combining defensive and offensive label propagation for core extraction. Phys Rev E Stat Nonlin Soft Matter Phys 2011, 83(3 Pt 2):036103. -->
<!-- 39 Traag VA, Waltman L, van Eck NJ: From Louvain to Leiden: guaranteeing well-connected communities. Sci Rep 2019, 9(1):5233. -->
<!-- 40 Wang B, Zhu J, Pierson E, Ramazzotti D, Batzoglou S: Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning. Nat Methods 2017, 14(4):414-416. -->
<!-- 41 Finak G, McDavid A, Yajima M, Deng J, Gersuk V, Shalek AK, Slichter CK, Miller HW, McElrath MJ, Prlic M et al: MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. Genome Biol 2015, 16:278. -->
<!-- 42 Kharchenko PV, Silberstein L, Scadden DT: Bayesian approach to single-cell differential expression analysis. Nat Methods 2014, 11(7):740-742. -->
<!-- 43 Miao Z, Deng K, Wang X, Zhang X: DEsingle for detecting three types of differential expression in single-cell RNA-seq data. Bioinformatics 2018, 34(18):3223-3224. -->
<!-- 44 Rosenberg AaH, J. : Proceedings of the 2007 joint conference on empirical methods in natural language processing and computational natural language learning (EMNLP-CoNLL). 2007:410-420. -->
<!-- 45 Hubert L, Arabie P: Comparing Partitions. Journal of Classification 1985, 2(2-3):193-218. -->
<!-- 46 Rousseeuw PJ: Silhouettes: a graphical aid to the interpretation and validation of cluster analysis. Journal of computational and applied mathematics 1987, 20:53-65. -->
<!-- 47 Strehl A, Ghosh J: Cluster ensembles---a knowledge reuse framework for combining multiple partitions. Journal of machine learning research 2002, 3(Dec):583-617. -->
<!-- 48 Hou W, Ji Z, Ji H, Hicks SC: A systematic evaluation of single-cell RNA-sequencing imputation methods. Genome Biol 2020, 21(1):218. -->
<!-- 49 Sun S, Zhu, J., Ma, Y. and Zhou, X.: Accuracy, robustness and scalability of dimensionality reduction methods for single-cell RNA-seq analysis. Genome Biol 2019, 20(1):269. -->
<!-- 50 Hubert LaA, P.: Comparing Partitions. Journal of Classification 1985, 2:193-218. -->
<!-- 51 Borgwardt KM, Gretton A, Rasch MJ, Kriegel HP, Scholkopf B, Smola AJ: Integrating structured biological data by Kernel Maximum Mean Discrepancy. Bioinformatics 2006, 22(14):e49-57. -->
<!-- 52 Buttner M, Miao Z, Wolf FA, Teichmann SA, Theis FJ: A test metric for assessing single-cell RNA-seq batch correction. Nat Methods 2019, 16(1):43-49. -->
<!-- 53 Cover TM: Elements of information theory: John Wiley & Sons; 1999. -->
<!-- 54 Rosenberg A, Hirschberg J: V-measure: A conditional entropy-based external cluster evaluation measure. In: Proceedings of the 2007 joint conference on empirical methods in natural language processing and computational natural language learning (EMNLP-CoNLL): 2007; 2007: 410-420. -->
<!-- 55 van Dijk D, Sharma R, Nainys J, Yim K, Kathail P, Carr AJ, Burdziak C, Moon KR, Chaffer CL, Pattabiraman D et al: Recovering Gene Interactions from Single-Cell Data Using Data Diffusion. Cell 2018, 174(3):716-729 e727. -->
<!-- 56 Wang J, Agarwal D, Huang M, Hu G, Zhou Z, Ye C, Zhang NR: Data denoising with transfer learning in single-cell transcriptomics. Nat Methods 2019, 16(9):875-878. -->
<!-- 57 Badsha MB, Li R, Liu B, Li YI, Xian M, Banovich NE, Fu AQ: Imputation of single-cell gene expression with an autoencoder neural network. Quant Biol 2020, 8(1):78-94. -->
<!-- 58 Yu B, Chen C, Qi R, Zheng R, Skillman-Lawrence PJ, Wang X, Ma A, Gu H: scGMAI: a Gaussian mixture model for clustering single-cell RNA-Seq data based on deep autoencoder. Brief Bioinform 2020. -->
<!-- 59 Lin P, Troup M, Ho JW: CIDR: Ultrafast and accurate clustering through imputation for single-cell RNA-seq data. Genome Biol 2017, 18(1):59. -->
<!-- 60 Berthelot D, Schumm, T. and Metz, L.: BEGAN: Boundary Equilibrium Generative Adversarial Networks. arXiv 2017. -->
<!-- 61 Wang T, Johnson TS, Shao W, Lu Z, Helm BR, Zhang J, Huang K: BERMUDA: a novel deep transfer learning method for single-cell RNA sequencing batch correction reveals hidden high-resolution cellular subtypes. Genome Biol 2019, 20(1):165. -->
<!-- 62 Crow M, Paul A, Ballouz S, Huang ZJ, Gillis J: Characterizing the replicability of cell types defined by single cell RNA-sequencing data using MetaNeighbor. Nat Commun 2018, 9(1):884. -->
<!-- 63 Polanski K, Young MD, Miao Z, Meyer KB, Teichmann SA, Park JE: BBKNN: fast batch alignment of single cell transcriptomes. Bioinformatics 2020, 36(3):964-965. -->
<!-- 64 Li X, Wang K, Lyu Y, Pan H, Zhang J, Stambolian D, Susztak K, Reilly MP, Hu G, Li M: Deep learning enables accurate clustering with batch effect removal in single-cell RNA-seq analysis. Nat Commun 2020, 11(1):2338. -->
<!-- 65 Guo X, Gao, L., Liu, X., and Yin, J.: Improved deep embedded clustering with local structure preservation. Proc 26th International Joint Conference on Artificial Integlligence 2017:1753-1759. -->
<!-- 66 Hie B, Bryson B, Berger B: Efficient integration of heterogeneous single-cell transcriptomes using Scanorama. Nat Biotechnol 2019, 37(6):685-691. -->
<!-- 67 Wang D, Hou S, Zhang L, Wang X, Liu B, Zhang Z: iMAP: integration of multiple single-cell datasets by adversarial paired transfer networks. Genome Biol 2021, 22(1):63. -->
<!-- 68 Lin C, Jain S, Kim H, Bar-Joseph Z: Using neural networks for reducing the dimensions of single-cell RNA-Seq data. Nucleic Acids Res 2017, 45(17):e156. -->
<!-- 69 Fowlkes EB, Mallows CL: A Method for Comparing Two Hierarchical Clusterings. Journal of the American Statistical Association 1983, 78(383):553-569. -->
<!-- 70 Rashid S, Shah S, Bar-Joseph Z, Pandya R: Dhaka: Variational Autoencoder for Unmasking Tumor Heterogeneity from Single Cell Genomic Data. Bioinformatics 2019. -->
<!-- 71 Tirosh I, Izar B, Prakadan SM, Wadsworth MH, 2nd, Treacy D, Trombetta JJ, Rotem A, Rodman C, Lian C, Murphy G et al: Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq. Science 2016, 352(6282):189-196. -->
<!-- 72 Zahn H, Steif A, Laks E, Eirew P, VanInsberghe M, Shah SP, Aparicio S, Hansen CL: Scalable whole-genome single-cell library preparation without preamplification. Nature Methods 2017, 14(2):167-173. -->
<!-- 73 Ding J, Condon A, Shah SP: Interpretable dimensionality reduction of single cell transcriptome data with deep generative models. Nat Commun 2018, 9(1):2002. -->
<!-- 74 van der Maaten LH, G.: Visualizing data using t-SNE. J Mach Learn 2008, 9:2579-2605. -->
<!-- 75 Gronbech CH, Vording MF, Timshel PN, Sonderby CK, Pers TH, Winther O: scVAE: variational auto-encoders for single-cell gene expression data. Bioinformatics 2020, 36(16):4415-4422. -->
<!-- 76 Wang D, Gu J: VASC: Dimension Reduction and Visualization of Single-cell RNA-seq Data by Deep Variational Autoencoder. Genomics Proteomics Bioinformatics 2018, 16(5):320-331. -->
<!-- 77 Jang E. GSaPB: Categorical reparameterization with gumbel-softmax. arXiv 2016. -->
<!-- 78 Tian T, Wan, J., Song, Q. et al.: Clustering single-cell RNA-seq data with a model-based deep learning approach. Nat Mach Intell 2019, 1. -->
<!-- 79 Regev A, Teichmann SA, Lander ES, Amit I, Benoist C, Birney E, Bodenmiller B, Campbell P, Carninci P, Clatworthy M et al: The Human Cell Atlas. Elife 2017, 6. -->
<!-- 80 Xie J, Girshick R, Farhadi A: Unsupervised deep embedding for clustering analysis. In: International conference on machine learning: 2016: PMLR; 2016: 478-487. -->
<!-- 81 La Manno G, Gyllborg D, Codeluppi S, Nishimura K, Salto C, Zeisel A, Borm LE, Stott SRW, Toledo EM, Villaescusa JC et al: Molecular Diversity of Midbrain Development in Mouse, Human, and Stem Cells. Cell 2016, 167(2):566-580 e519. -->
<!-- 82 Marouf M, Machart P, Bansal V, Kilian C, Magruder DS, Krebs CF, Bonn S: Realistic in silico generation and augmentation of single-cell RNA-seq data using generative adversarial networks. Nat Commun 2020, 11(1):166. -->
<!-- 83 Miyato TaK, M: cGANs with projection discriminator. Preprint 2018. -->
<!-- 84 Aibar S, Gonzalez-Blas CB, Moerman T, Huynh-Thu VA, Imrichova H, Hulselmans G, Rambow F, Marine JC, Geurts P, Aerts J et al: SCENIC: single-cell regulatory network inference and clustering. Nat Methods 2017, 14(11):1083-1086. -->
<!-- 85 Zappia L, Phipson B, Oshlack A: Splatter: simulation of single-cell RNA sequencing data. Genome Biol 2017, 18(1):174. -->
<!-- 86 Lindenbaum O, Stanley, J. S., Wolf, G. and Krishnaswamy, S. : Geometry-based data generation. Advances in Neural Information Processing Systems 2018. -->
<!-- 87 Svensson V, Gayoso A, Yosef N, Pachter L: Interpretable factor models of single-cell RNA-seq via variational autoencoders. Bioinformatics 2020, 36(11):3418-3421. -->
<!-- 88 Levine JH, Simonds EF, Bendall SC, Davis KL, Amir el AD, Tadmor MD, Litvin O, Fienberg HG, Jager A, Zunder ER et al: Data-Driven Phenotypic Dissection of AML Reveals Progenitor-like Cells that Correlate with Prognosis. Cell 2015, 162(1):184-197. -->
<!-- 89 Qiu X, Mao Q, Tang Y, Wang L, Chawla R, Pliner HA, Trapnell C: Reversed graph embedding resolves complex single-cell trajectories. Nat Methods 2017, 14(10):979-982. -->
<!-- 90 McInnes L, Healy, J. & Melville, J.: Umap: uniform manifold approximation and projection for dimension reduction. ArXiv 2018. -->
<!-- 91 Moon KRea: PHATE: a dimensionality reduction method for visualizing trajectory structures in high-dimensional biological data. bioRxiv 2017. -->
<!-- 92 Deng Y, Bao F, Dai Q, Wu LF, Altschuler SJ: Scalable analysis of cell-type composition from single-cell transcriptomics using deep recurrent learning. Nat Methods 2019, 16(4):311-314. -->
<!-- 93 Bernstein NJ, Fong NL, Lam I, Roy MA, Hendrickson DG, Kelley DR: Solo: Doublet Identification in Single-Cell RNA-Seq via Semi-Supervised Deep Learning. Cell Syst 2020, 11(1):95-101 e105. -->
<!-- 94 Wolock SL, Lopez R, Klein AM: Scrublet: Computational Identification of Cell Doublets in Single-Cell Transcriptomic Data. Cell Syst 2019, 8(4):281-291 e289. -->
<!-- 95 McGinnis CS, Murrow LM, Gartner ZJ: DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors. Cell Syst 2019, 8(4):329-337 e324. -->
<!-- 96 Torroja C, Sanchez-Cabo F: Digitaldlsorter: Deep-Learning on scRNA-Seq to Deconvolute Gene Expression Data. Front Genet 2019, 10:978. -->
<!-- 97 Chung W, Eum HH, Lee HO, Lee KM, Lee HB, Kim KT, Ryu HS, Kim S, Lee JE, Park YH et al: Single-cell RNA-seq enables comprehensive tumour and immune cell profiling in primary breast cancer. Nat Commun 2017, 8:15081. -->
<!-- 98 Li H, Courtois ET, Sengupta D, Tan Y, Chen KH, Goh JJL, Kong SL, Chua C, Hon LK, Tan WS et al: Reference component analysis of single-cell transcriptomes elucidates cellular heterogeneity in human colorectal tumors. Nat Genet 2017, 49(5):708-718. -->
<!-- 99 Aran D, Hu Z, Butte AJ: xCell: digitally portraying the tissue cellular heterogeneity landscape. Genome Biol 2017, 18(1):220. -->
<!-- 100    Yoshihara K, Shahmoradgoli M, Martinez E, Vegesna R, Kim H, Torres-Garcia W, Trevino V, Shen H, Laird PW, Levine DA et al: Inferring tumour purity and stromal and immune cell admixture from expression data. Nat Commun 2013, 4:2612. -->
<!-- 101    Racle J, de Jonge K, Baumgaertner P, Speiser DE, Gfeller D: Simultaneous enumeration of cancer and immune cell types from bulk tumor gene expression data. Elife 2017, 6. -->
<!-- 102    Becht E, Giraldo NA, Lacroix L, Buttard B, Elarouci N, Petitprez F, Selves J, Laurent-Puig P, Sautes-Fridman C, Fridman WH et al: Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression. Genome Biol 2016, 17(1):218. -->
<!-- 103    Wang L, Nie, R., Yu, Z. et al.: An interpretable deep-learning architecture of capsule networks for identifying cell-type gene expression programs from single-cell RNA-sequencing data. Nat Mach Intell 2020, 2:693-703. -->
<!-- 104    Patel ND, Nguang SK, Coghill GG: Neural network implementation using bit streams. IEEE Trans Neural Netw 2007, 18(5):1488-1504. -->
<!-- 105    Zheng GX, Terry JM, Belgrader P, Ryvkin P, Bent ZW, Wilson R, Ziraldo SB, Wheeler TD, McDermott GP, Zhu J et al: Massively parallel digital transcriptional profiling of single cells. Nat Commun 2017, 8:14049. -->
<!-- 106    Shekhar K, Lapan SW, Whitney IE, Tran NM, Macosko EZ, Kowalczyk M, Adiconis X, Levin JZ, Nemesh J, Goldman M et al: Comprehensive Classification of Retinal Bipolar Neurons by Single-Cell Transcriptomics. Cell 2016, 166(5):1308-1323 e1330. -->
<!-- 107    Dong Z, Alterovitz G: netAE: semi-supervised dimensionality reduction of single-cell RNA sequencing to facilitate cell labeling. Bioinformatics 2021, 37(1):43-49. -->
<!-- 108    Newman ME: Modularity and community structure in networks. Proc Natl Acad Sci U S A 2006, 103(23):8577-8582. -->
<!-- 109    Xu C, Lopez R, Mehlman E, Regier J, Jordan MI, Yosef N: Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models. Mol Syst Biol 2021, 17(1):e9620. -->
<!-- 110    Ge S, Wang H, Alavi A, Xing E, Bar-Joseph Z: Supervised Adversarial Alignment of Single-Cell RNA-seq Data. J Comput Biol 2021, 28(5):501-513. -->
<!-- 111    Hadsell R, Chopra S, LeCun Y: Dimensionality reduction by learning an invariant mapping. In: 2006 IEEE Computer Society Conference on Computer Vision and Pattern Recognition (CVPR'06): 2006: IEEE; 2006: 1735-1742. -->
<!-- 112    Lieberman Y, Rokach L, Shay T: CaSTLe–classification of single cells by transfer learning: harnessing the power of publicly available single cell RNA sequencing experiments to annotate new experiments. PloS one 2018, 13(10):e0205499. -->
<!-- 113    Yuan Y, Bar-Joseph Z: Deep learning for inferring gene relationships from single-cell expression data. Proc Natl Acad Sci U S A 2019. -->
<!-- 114    Alavi A, Ruffalo M, Parvangada A, Huang Z, Bar-Joseph Z: A web server for comparative analysis of single-cell RNA-seq data. Nat Commun 2018, 9(1):4768. -->
<!-- 115    Yevshin I, Sharipov R, Valeev T, Kel A, Kolpakov F: GTRD: a database of transcription factor binding sites identified by ChIP-seq experiments. Nucleic Acids Research 2016, 45(D1):D61-D67. -->
<!-- 116    Wang YXR, Waterman MS, Huang HY: Gene coexpression measures in large heterogeneous samples using count statistics. Proceedings of the National Academy of Sciences of the United States of America 2014, 111(46):16371-16376. -->
<!-- 117    Krishnaswamy S, Spitzer MH, Mingueneau M, Bendall SC, Litvin O, Stone E, Pe'er D, Nolan GP: Systems biology. Conditional density-based analysis of T cell signaling in single-cell data. Science 2014, 346(6213):1250689. -->
<!-- 118    Kanehisa M, Furumichi M, Tanabe M, Sato Y, Morishima K: KEGG: new perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Res 2017, 45(D1):D353-D361. -->
<!-- 119    Fabregat A, Jupe S, Matthews L, Sidiropoulos K, Gillespie M, Garapati P, Haw R, Jassal B, Korninger F, May B et al: The Reactome Pathway Knowledgebase. Nucleic Acids Res 2018, 46(D1):D649-D655. -->
<!-- 120    Huynh-Thu VA, Irrthum A, Wehenkel L, Geurts P: Inferring regulatory networks from expression data using tree-based methods. PloS one 2010, 5(9):e12776. -->
<!-- 121    Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, Paulovich A, Pomeroy SL, Golub TR, Lander ES et al: Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A 2005, 102(43):15545-15550. -->
<!-- 122    Lotfollahi M, Wolf FA, Theis FJ: scGen predicts single-cell perturbation responses. Nat Methods 2019, 16(8):715-721. -->
<!-- 123    Duvenaud D, Maclaurin D, Iparraguirre J, Bombarell R, Hirzel T, Aspuru-Guzik A, Adams R: Advances in Neural Information Processing Systems 28. Cortes C, Lawrence ND, Lee DD, Sugiyama M, Garnett R, Eds 2015:2224-2232. -->
<!-- 124    Amodio M, Krishnaswamy S: MAGAN: Aligning biological manifolds. In: International Conference on Machine Learning: 2018: PMLR; 2018: 215-223. -->
<!-- 125    Kang HM, Subramaniam M, Targ S, Nguyen M, Maliskova L, McCarthy E, Wan E, Wong S, Byrnes L, Lanata CM et al: Multiplexed droplet single-cell RNA-sequencing using natural genetic variation. Nat Biotechnol 2018, 36(1):89-94. -->
<!-- 126    Haber AL, Biton M, Rogel N, Herbst RH, Shekhar K, Smillie C, Burgin G, Delorey TM, Howitt MR, Katz Y et al: A single-cell survey of the small intestinal epithelium. Nature 2017, 551(7680):333-339. -->
<!-- 127    Hagai T, Chen X, Miragaia RJ, Rostom R, Gomes T, Kunowska N, Henriksson J, Park JE, Proserpio V, Donati G et al: Gene expression variability across cells and species shapes innate immunity. Nature 2018, 563(7730):197-202. -->
<!-- 128    Peng Y, Baulier E, Ke Y, Young A, Ahmedli NB, Schwartz SD, Farber DB: Human embryonic stem cells extracellular vesicles and their effects on immortalized human retinal Muller cells. PLoS One 2018, 13(3):e0194004. -->
<!-- 129    Stoeckius M, Hafemeister C, Stephenson W, Houck-Loomis B, Chattopadhyay PK, Swerdlow H, Satija R, Smibert P: Simultaneous epitope and transcriptome measurement in single cells. Nat Methods 2017, 14(9):865-868. -->
<!-- 130    Azizi E, Carr AJ, Plitas G, Cornish AE, Konopacki C, Prabhakaran S, Nainys J, Wu K, Kiseliovas V, Setty M et al: Single-Cell Map of Diverse Immune Phenotypes in the Breast Tumor Microenvironment. Cell 2018, 174(5):1293-1308 e1236. -->
<!-- 131    Chu LF, Leng N, Zhang J, Hou Z, Mamott D, Vereide DT, Choi J, Kendziorski C, Stewart R, Thomson JA: Single-cell RNA-seq reveals novel regulators of human embryonic stem cell differentiation to definitive endoderm. Genome Biol 2016, 17(1):173. -->
<!-- 132    Baron M, Veres A, Wolock SL, Faust AL, Gaujoux R, Vetere A, Ryu JH, Wagner BK, Shen-Orr SS, Klein AM et al: A Single-Cell Transcriptomic Map of the Human and Mouse Pancreas Reveals Inter- and Intra-cell Population Structure. Cell Syst 2016, 3(4):346-360 e344. -->
<!-- 133    Camp JG, Sekine K, Gerber T, Loeffler-Wirth H, Binder H, Gac M, Kanton S, Kageyama J, Damm G, Seehofer D et al: Multilineage communication regulates human liver bud development from pluripotency. Nature 2017, 546(7659):533-538. -->
<!-- 134    Muraro MJ, Dharmadhikari G, Grun D, Groen N, Dielen T, Jansen E, van Gurp L, Engelse MA, Carlotti F, de Koning EJ et al: A Single-Cell Transcriptome Atlas of the Human Pancreas. Cell Syst 2016, 3(4):385-394 e383. -->
<!-- 135    Darmanis S, Sloan SA, Zhang Y, Enge M, Caneda C, Shuer LM, Hayden Gephart MG, Barres BA, Quake SR: A survey of human brain transcriptome diversity at the single cell level. Proc Natl Acad Sci U S A 2015, 112(23):7285-7290. -->
<!-- 136    Tirosh I, Venteicher AS, Hebert C, Escalante LE, Patel AP, Yizhak K, Fisher JM, Rodman C, Mount C, Filbin MG et al: Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Nature 2016, 539(7628):309-313. -->
<!-- 137    Patel AP, Tirosh I, Trombetta JJ, Shalek AK, Gillespie SM, Wakimoto H, Cahill DP, Nahed BV, Curry WT, Martuza RL et al: Single-cell RNA-seq highlights intratumoral heterogeneity in primary glioblastoma. Science 2014, 344(6190):1396-1401. -->
<!-- 138    Zahn H, Steif A, Laks E, Eirew P, VanInsberghe M, Shah SP, Aparicio S, Hansen CL: Scalable whole-genome single-cell library preparation without preamplification. Nat Methods 2017, 14(2):167-173. -->
<!-- 139    Pollen AA, Nowakowski TJ, Shuga J, Wang X, Leyrat AA, Lui JH, Li N, Szpankowski L, Fowler B, Chen P et al: Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and activated signaling pathways in developing cerebral cortex. Nat Biotechnol 2014, 32(10):1053-1058. -->
<!-- 140    Xin Y, Kim J, Okamoto H, Ni M, Wei Y, Adler C, Murphy AJ, Yancopoulos GD, Lin C, Gromada J: RNA Sequencing of Single Human Islet Cells Reveals Type 2 Diabetes Genes. Cell Metab 2016, 24(4):608-615. -->
<!-- 141    Yan L, Yang M, Guo H, Yang L, Wu J, Li R, Liu P, Lian Y, Zheng X, Yan J et al: Single-cell RNA-Seq profiling of human preimplantation embryos and embryonic stem cells. Nat Struct Mol Biol 2013, 20(9):1131-1139. -->
<!-- 142    Chevrier S, Levine JH, Zanotelli VRT, Silina K, Schulz D, Bacac M, Ries CH, Ailles L, Jewett MAS, Moch H et al: An Immune Atlas of Clear Cell Renal Cell Carcinoma. Cell 2017, 169(4):736-749 e718. -->
<!-- 143    McGinnis CS, Patterson DM, Winkler J, Conrad DN, Hein MY, Srivastava V, Hu JL, Murrow LM, Weissman JS, Werb Z et al: MULTI-seq: sample multiplexing for single-cell RNA sequencing using lipid-tagged indices. Nat Methods 2019, 16(7):619-626. -->
<!-- 144    Han X, Wang R, Zhou Y, Fei L, Sun H, Lai S, Saadatpour A, Zhou Z, Chen H, Ye F et al: Mapping the Mouse Cell Atlas by Microwell-Seq. Cell 2018, 172(5):1091-1107 e1017. -->
<!-- 145    Hrvatin S, Hochbaum DR, Nagy MA, Cicconet M, Robertson K, Cheadle L, Zilionis R, Ratner A, Borges-Monroy R, Klein AM et al: Single-cell analysis of experience-dependent transcriptomic states in the mouse visual cortex. Nat Neurosci 2018, 21(1):120-129. -->
<!-- 146    Joost S, Zeisel A, Jacob T, Sun X, La Manno G, Lonnerberg P, Linnarsson S, Kasper M: Single-Cell Transcriptomics Reveals that Differentiation and Spatial Signatures Shape Epidermal and Hair Follicle Heterogeneity. Cell Syst 2016, 3(3):221-237 e229. -->
<!-- 147    Paul F, Arkin Y, Giladi A, Jaitin DA, Kenigsberg E, Keren-Shaul H, Winter D, Lara-Astiaso D, Gury M, Weiner A et al: Transcriptional Heterogeneity and Lineage Commitment in Myeloid Progenitors. Cell 2015, 163(7):1663-1677. -->
<!-- 148    Buettner F, Natarajan KN, Casale FP, Proserpio V, Scialdone A, Theis FJ, Teichmann SA, Marioni JC, Stegle O: Computational analysis of cell-to-cell heterogeneity in single-cell RNA-sequencing data reveals hidden subpopulations of cells. Nat Biotechnol 2015, 33(2):155-160. -->
<!-- 149    Biase FH, Cao X, Zhong S: Cell fate inclination within 2-cell and 4-cell mouse embryos revealed by single-cell RNA sequencing. Genome Res 2014, 24(11):1787-1796. -->
<!-- 150    Deng Q, Ramskold D, Reinius B, Sandberg R: Single-cell RNA-seq reveals dynamic, random monoallelic gene expression in mammalian cells. Science 2014, 343(6167):193-196. -->
<!-- 151    Klein AM, Mazutis L, Akartuna I, Tallapragada N, Veres A, Li V, Peshkin L, Weitz DA, Kirschner MW: Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells. Cell 2015, 161(5):1187-1201. -->
<!-- 152    Goolam M, Scialdone A, Graham SJL, Macaulay IC, Jedrusik A, Hupalowska A, Voet T, Marioni JC, Zernicka-Goetz M: Heterogeneity in Oct4 and Sox2 Targets Biases Cell Fate in 4-Cell Mouse Embryos. Cell 2016, 165(1):61-74. -->
<!-- 153    Kim JK, Kolodziejczyk AA, Ilicic T, Teichmann SA, Marioni JC: Characterizing noise structure in single-cell RNA-seq distinguishes genuine from technical stochastic allelic expression. Nat Commun 2015, 6:8687. -->
<!-- 154    Usoskin D, Furlan A, Islam S, Abdo H, Lonnerberg P, Lou D, Hjerling-Leffler J, Haeggstrom J, Kharchenko O, Kharchenko PV et al: Unbiased classification of sensory neuron types by large-scale single-cell RNA sequencing. Nat Neurosci 2015, 18(1):145-153. -->
<!-- 155    Zeisel A, Munoz-Manchado AB, Codeluppi S, Lonnerberg P, La Manno G, Jureus A, Marques S, Munguba H, He L, Betsholtz C et al: Brain structure. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq. Science 2015, 347(6226):1138-1142. -->
<!-- 156    Yu Z, Liao J, Chen Y, Zou C, Zhang H, Cheng J, Liu D, Li T, Zhang Q, Li J et al: Single-Cell Transcriptomic Map of the Human and Mouse Bladders. J Am Soc Nephrol 2019, 30(11):2159-2176. -->
<!-- 157    Tusi BK, Wolock SL, Weinreb C, Hwang Y, Hidalgo D, Zilionis R, Waisman A, Huh JR, Klein AM, Socolovsky M: Population snapshots predict early haematopoietic and erythroid hierarchies. Nature 2018, 555(7694):54-60. -->
<!-- 158    Pijuan-Sala B, Griffiths JA, Guibentif C, Hiscock TW, Jawaid W, Calero-Nieto FJ, Mulas C, Ibarra-Soria X, Tyser RCV, Ho DLL et al: A single-cell molecular map of mouse gastrulation and early organogenesis. Nature 2019, 566(7745):490-495. -->
<!-- 159    Cao J, Spielmann M, Qiu X, Huang X, Ibrahim DM, Hill AJ, Zhang F, Mundlos S, Christiansen L, Steemers FJ et al: The single-cell transcriptional landscape of mammalian organogenesis. Nature 2019, 566(7745):496-502. -->
<!-- 160    Setty M, Tadmor MD, Reich-Zeliger S, Angel O, Salame TM, Kathail P, Choi K, Bendall S, Friedman N, Pe'er D: Wishbone identifies bifurcating developmental trajectories from single-cell data. Nat Biotechnol 2016, 34(6):637-645. -->
<!-- 161    Nestorowa S, Hamey FK, Pijuan Sala B, Diamanti E, Shepherd M, Laurenti E, Wilson NK, Kent DG, Gottgens B: A single-cell resolution map of mouse hematopoietic stem and progenitor cell differentiation. Blood 2016, 128(8):e20-31. -->
<!-- 162    Cao J, Packer JS, Ramani V, Cusanovich DA, Huynh C, Daza R, Qiu X, Lee C, Furlan SN, Steemers FJ et al: Comprehensive single-cell transcriptional profiling of a multicellular organism. Science 2017, 357(6352):661-667. -->

</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN76" class="csl-entry">
Amodio, M., D. van Dijk, K. Srinivasan, W. S. Chen, H. Mohsen, K. R. Moon, A. Campbell, et al. 2019. <span>“Exploring Single-Cell Data with Deep Multitasking Neural Networks.”</span> Journal Article. <em>Nat Methods</em> 16 (11): 1139–45. <a href="https://doi.org/10.1038/s41592-019-0576-7">https://doi.org/10.1038/s41592-019-0576-7</a>.
</div>
<div id="ref-RN78" class="csl-entry">
Bost, P., A. Giladi, Y. Liu, Y. Bendjelal, G. Xu, E. David, R. Blecher-Gonen, et al. 2020. <span>“Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients.”</span> Journal Article. <em>Cell</em> 181 (7): 1475–1488 e12. <a href="https://doi.org/10.1016/j.cell.2020.05.006">https://doi.org/10.1016/j.cell.2020.05.006</a>.
</div>
<div id="ref-RN85" class="csl-entry">
Kinker, G. S., A. C. Greenwald, R. Tal, Z. Orlova, M. S. Cuoco, J. M. McFarland, A. Warren, et al. 2020. <span>“Pan-Cancer Single-Cell RNA-Seq Identifies Recurring Programs of Cellular Heterogeneity.”</span> Journal Article. <em>Nat Genet</em> 52 (11): 1208–18. <a href="https://doi.org/10.1038/s41588-020-00726-6">https://doi.org/10.1038/s41588-020-00726-6</a>.
</div>
<div id="ref-RN87" class="csl-entry">
Lahnemann, D., J. Koster, E. Szczurek, D. J. McCarthy, S. C. Hicks, M. D. Robinson, C. A. Vallejos, et al. 2020. <span>“Eleven Grand Challenges in Single-Cell Data Science.”</span> Journal Article. <em>Genome Biol</em> 21 (1): 31. <a href="https://doi.org/10.1186/s13059-020-1926-6">https://doi.org/10.1186/s13059-020-1926-6</a>.
</div>
<div id="ref-RN88" class="csl-entry">
Lopez, R., J. Regier, M. B. Cole, M. I. Jordan, and N. Yosef. 2018. <span>“Deep Generative Modeling for Single-Cell Transcriptomics.”</span> Journal Article. <em>Nat Methods</em> 15 (12): 1053–58. <a href="https://doi.org/10.1038/s41592-018-0229-2">https://doi.org/10.1038/s41592-018-0229-2</a>.
</div>
<div id="ref-RN91" class="csl-entry">
Mannarapu, M., B. Dariya, and O. R. Bandapalli. 2021. <span>“Application of Single-Cell Sequencing Technologies in Pancreatic Cancer.”</span> Journal Article. <em>Mol Cell Biochem</em> 476 (6): 2429–37. <a href="https://doi.org/10.1007/s11010-021-04095-4">https://doi.org/10.1007/s11010-021-04095-4</a>.
</div>
<div id="ref-RN59" class="csl-entry">
Navin, N. E. 2015. <span>“The First Five Years of Single-Cell Cancer Genomics and Beyond.”</span> Journal Article. <em>Genome Res</em> 25 (10): 1499–1507. <a href="https://doi.org/10.1101/gr.191098.115">https://doi.org/10.1101/gr.191098.115</a>.
</div>
<div id="ref-RN95" class="csl-entry">
Srinivasan, S., A. Leshchyk, N. T. Johnson, and D. Korkin. 2020. <span>“A Hybrid Deep Clustering Approach for Robust Cell Type Profiling Using Single-Cell RNA-Seq Data.”</span> Journal Article. <em>RNA</em> 26 (10): 1303–19. <a href="https://doi.org/10.1261/rna.074427.119">https://doi.org/10.1261/rna.074427.119</a>.
</div>
<div id="ref-RN96" class="csl-entry">
Stuart, T., A. Butler, P. Hoffman, C. Hafemeister, E. Papalexi, 3rd Mauck W. M., Y. Hao, M. Stoeckius, P. Smibert, and R. Satija. 2019. <span>“Comprehensive Integration of Single-Cell Data.”</span> Journal Article. <em>Cell</em> 177 (7): 1888–1902 e21. <a href="https://doi.org/10.1016/j.cell.2019.05.031">https://doi.org/10.1016/j.cell.2019.05.031</a>.
</div>
<div id="ref-RN98" class="csl-entry">
Vitak, S. A., K. A. Torkenczy, J. L. Rosenkrantz, A. J. Fields, L. Christiansen, M. H. Wong, L. Carbone, F. J. Steemers, and A. Adey. 2017. <span>“Sequencing Thousands of Single-Cell Genomes with Combinatorial Indexing.”</span> Journal Article. <em>Nat Methods</em> 14 (3): 302–8. <a href="https://doi.org/10.1038/nmeth.4154">https://doi.org/10.1038/nmeth.4154</a>.
</div>
<div id="ref-RN99" class="csl-entry">
Wauters, E., P. Van Mol, A. D. Garg, S. Jansen, Y. Van Herck, L. Vanderbeke, A. Bassez, et al. 2021. <span>“Discriminating Mild from Critical COVID-19 by Innate and Adaptive Immune Single-Cell Profiling of Bronchoalveolar Lavages.”</span> Journal Article. <em>Cell Res</em> 31 (3): 272–90. <a href="https://doi.org/10.1038/s41422-020-00455-9">https://doi.org/10.1038/s41422-020-00455-9</a>.
</div>
<div id="ref-RN100" class="csl-entry">
Wolf, F. A., P. Angerer, and F. J. Theis. 2018. <span>“SCANPY: Large-Scale Single-Cell Gene Expression Data Analysis.”</span> Journal Article. <em>Genome Biol</em> 19 (1): 15. <a href="https://doi.org/10.1186/s13059-017-1382-0">https://doi.org/10.1186/s13059-017-1382-0</a>.
</div>
<div id="ref-RN191" class="csl-entry">
Wolock, S. L., R. Lopez, and A. M. Klein. 2019. <span>“Scrublet: Computational Identification of Cell Doublets in Single-Cell Transcriptomic Data.”</span> Journal Article. <em>Cell Syst</em> 8 (4): 281–291 e9. <a href="https://doi.org/10.1016/j.cels.2018.11.005">https://doi.org/10.1016/j.cels.2018.11.005</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="about-this-book.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch-2.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/Huang-AI4Medicine-Lab/survey-of-DL-for-scRNA-seq-analysis/edit/main/01-introduction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["survey-of-DL-for-scRNA-seq-analysis.pdf", "survey-of-DL-for-scRNA-seq-analysis.epub"],
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
