<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis</title>
  <meta name="description" content="7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  <meta name="generator" content="bookdown 0.23 and GitBook 2.6.7" />

  <meta property="og:title" content="7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="Huang-AI4Medicine-Lab/survey-of-DL-for-scRNA-seq-analysis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  
  
  

<meta name="author" content="Mario Flores1§, Zhentao Liu1, Tinghe Zhang1, Md Musaddaqui Hasib1, Yu-Chiao Chiu2, Zhenqing Ye2,3, Karla Paniagua1, Sumin Jo1, Jianqiu Zhang1, Shou-Jiang Gao4,6, Yufang Jin1, Yidong Chen2,3§, and Yufei Huang5,6§" />


<meta name="date" content="2021-09-16" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="ch-6.html"/>

<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Survey of Deep Learning for scRNA-seq Analysis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About the authors</a></li>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html"><i class="fa fa-check"></i>About this book</a>
<ul>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html#key-points"><i class="fa fa-check"></i>Key Points</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="ch-1.html"><a href="ch-1.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="ch-2.html"><a href="ch-2.html"><i class="fa fa-check"></i><b>2</b> Overview of scRNA-seq processing pipeline</a></li>
<li class="chapter" data-level="3" data-path="ch-3.html"><a href="ch-3.html"><i class="fa fa-check"></i><b>3</b> Overview of common deep learning models for scRNA-seq analysis</a>
<ul>
<li class="chapter" data-level="3.1" data-path="ch-3.html"><a href="ch-3.html#ch-3-1"><i class="fa fa-check"></i><b>3.1</b> Variational Autoencoder (VAEs) for scRNA-Seq data</a></li>
<li class="chapter" data-level="3.2" data-path="ch-3.html"><a href="ch-3.html#ch-3-2"><i class="fa fa-check"></i><b>3.2</b> Autoencoders (AEs) for scRNA-seq data</a></li>
<li class="chapter" data-level="3.3" data-path="ch-3.html"><a href="ch-3.html#ch-3-3"><i class="fa fa-check"></i><b>3.3</b> Generative adversarial networks (GANs) for scRNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-4.html"><a href="ch-4.html"><i class="fa fa-check"></i><b>4</b> Overview of datasets and evaluation metrics</a>
<ul>
<li class="chapter" data-level="4.1" data-path="ch-4.html"><a href="ch-4.html#ch-4-1"><i class="fa fa-check"></i><b>4.1</b> Evaluation methods</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-1"><i class="fa fa-check"></i><b>4.1.1</b> Imputation</a></li>
<li class="chapter" data-level="4.1.2" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-2"><i class="fa fa-check"></i><b>4.1.2</b> Batch effect correction</a></li>
<li class="chapter" data-level="4.1.3" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-3"><i class="fa fa-check"></i><b>4.1.3</b> Clustering</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-5.html"><a href="ch-5.html"><i class="fa fa-check"></i><b>5</b> Survey of deep learning models for scRNA-Seq analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-1"><i class="fa fa-check"></i><b>5.1</b> Imputation</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-1"><i class="fa fa-check"></i><b>5.1.1</b> DCA:deep count autoencoder</a></li>
<li class="chapter" data-level="5.1.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-2"><i class="fa fa-check"></i><b>5.1.2</b> SAVER-X: single-cell analysis via expression recovery harnessing external data</a></li>
<li class="chapter" data-level="5.1.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-3"><i class="fa fa-check"></i><b>5.1.3</b> DeepImpute (Deep neural network Imputation)</a></li>
<li class="chapter" data-level="5.1.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-4"><i class="fa fa-check"></i><b>5.1.4</b> LATE: Learning with AuToEncoder</a></li>
<li class="chapter" data-level="5.1.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-5"><i class="fa fa-check"></i><b>5.1.5</b> scGMAI</a></li>
<li class="chapter" data-level="5.1.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-6"><i class="fa fa-check"></i><b>5.1.6</b> scIGANs</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-2"><i class="fa fa-check"></i><b>5.2</b> Batch effect correction</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-1"><i class="fa fa-check"></i><b>5.2.1</b> BERMUDA: Batch Effect ReMoval Using Deep Autoencoders</a></li>
<li class="chapter" data-level="5.2.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-2"><i class="fa fa-check"></i><b>5.2.2</b> DESC: batch correction based on clustering</a></li>
<li class="chapter" data-level="5.2.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-3"><i class="fa fa-check"></i><b>5.2.3</b> iMAP: Integration of Multiple single-cell datasets by Adversarial Paired-style transfer networks</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-3"><i class="fa fa-check"></i><b>5.3</b> Dimension reduction, latent representation, clustering, and data augmentation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-1"><i class="fa fa-check"></i><b>5.3.1</b> Dimension reduction by AEs with gene-interaction constrained architecture</a></li>
<li class="chapter" data-level="5.3.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-2"><i class="fa fa-check"></i><b>5.3.2</b> Dhaka: a VAE-based dimension reduction model</a></li>
<li class="chapter" data-level="5.3.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-3"><i class="fa fa-check"></i><b>5.3.3</b> cvis: a VAE for capturing low-dimensional structures</a></li>
<li class="chapter" data-level="5.3.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-4"><i class="fa fa-check"></i><b>5.3.4</b> scVAE: VAE for single-cell gene expression data</a></li>
<li class="chapter" data-level="5.3.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-5"><i class="fa fa-check"></i><b>5.3.5</b> VASC: VAE for scRNA-seq</a></li>
<li class="chapter" data-level="5.3.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-6"><i class="fa fa-check"></i><b>5.3.6</b> scDeepCluster</a></li>
<li class="chapter" data-level="5.3.7" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-7"><i class="fa fa-check"></i><b>5.3.7</b> cscGAN: Conditional single-cell generative adversarial neural networks</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-4"><i class="fa fa-check"></i><b>5.4</b> Multi-functional models</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-1"><i class="fa fa-check"></i><b>5.4.1</b> scVI: single-cell variational inference</a></li>
<li class="chapter" data-level="5.4.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-2"><i class="fa fa-check"></i><b>5.4.2</b> LDVAE: linearly decoded variational autoencoder</a></li>
<li class="chapter" data-level="5.4.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-3"><i class="fa fa-check"></i><b>5.4.3</b> SAUCIE</a></li>
<li class="chapter" data-level="5.4.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-4"><i class="fa fa-check"></i><b>5.4.4</b> scScope</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-5"><i class="fa fa-check"></i><b>5.5</b> Doublet Classification</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-5-1"><i class="fa fa-check"></i><b>5.5.1</b> Solo</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-6"><i class="fa fa-check"></i><b>5.6</b> Automated cell type identification</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-1"><i class="fa fa-check"></i><b>5.6.1</b> DigitalDLSorter</a></li>
<li class="chapter" data-level="5.6.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-2"><i class="fa fa-check"></i><b>5.6.2</b> scCapsNet</a></li>
<li class="chapter" data-level="5.6.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-3"><i class="fa fa-check"></i><b>5.6.3</b> netAE: network-enhanced autoencoder</a></li>
<li class="chapter" data-level="5.6.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-4"><i class="fa fa-check"></i><b>5.6.4</b> scDGN - supervised adversarial alignment of single-cell RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="ch-5.html"><a href="ch-5.html#ch-5-7"><i class="fa fa-check"></i><b>5.7</b> Biological function prediction</a>
<ul>
<li class="chapter" data-level="5.7.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-7-1"><i class="fa fa-check"></i><b>5.7.1</b> CNNC: convolutional neural network for coexpression</a></li>
<li class="chapter" data-level="5.7.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-7-2"><i class="fa fa-check"></i><b>5.7.2</b> scGen, a generative model to predict perturbation response of single cells across cell types</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="ch-6.html"><a href="ch-6.html"><i class="fa fa-check"></i><b>6</b> Conclusion and Discussions</a></li>
<li class="chapter" data-level="7" data-path="ch-7.html"><a href="ch-7.html"><i class="fa fa-check"></i><b>7</b> Tables</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch-7" class="section level1" number="7">
<h1><span class="header-section-number">7</span> Tables</h1>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Table1"></span>
<img src="Table/Table1.png" alt="Deep Learning algorithms reviewed in the paper" width="850" />
<p class="caption">
Figure 7.1: Deep Learning algorithms reviewed in the paper
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Table2a"></span>
<img src="Table/Table2a.png" alt="Simulated single-cell data/algorithms" width="720" />
<p class="caption">
Figure 7.2: Simulated single-cell data/algorithms
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Table2b"></span>
<img src="Table/Table2b.png" alt="Human single-cell data sources used by different DL algorithms" width="712" />
<p class="caption">
Figure 7.3: Human single-cell data sources used by different DL algorithms
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Table2c"></span>
<img src="Table/Table2c.png" alt="Mouse single-cell data sources used by different DL algorithms" width="685" />
<p class="caption">
Figure 7.4: Mouse single-cell data sources used by different DL algorithms
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Table2d"></span>
<img src="Table/Table2d.png" alt="Single-cell data derived from other species" width="702" />
<p class="caption">
Figure 7.5: Single-cell data derived from other species
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Table2e"></span>
<img src="Table/Table2e.png" alt="Large single-cell data source used by various algorithms" width="698" />
<p class="caption">
Figure 7.6: Large single-cell data source used by various algorithms
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:Table3"></span>
<img src="Table/Table3.png" alt="Evaluation metrics used in surveyed DL algorithms" width="1419" />
<p class="caption">
Figure 7.7: Evaluation metrics used in surveyed DL algorithms
</p>
</div>

<div id="refs" class="references csl-bib-body hanging-indent">
<div class="csl-entry">
Abdelaal, T., L. Michielsen, D. Cats, D. Hoogduin, H. Mei, M. J. T. Reinders, and A. Mahfouz. 2019. <span>“A Comparison of Automatic Cell Identification Methods for Single-Cell RNA Sequencing Data.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 194. <a href="https://doi.org/10.1186/s13059-019-1795-z">https://doi.org/10.1186/s13059-019-1795-z</a>.
</div>
<div class="csl-entry">
Amodio, M., D. van Dijk, K. Srinivasan, W. S. Chen, H. Mohsen, K. R. Moon, A. Campbell, et al. 2019. <span>“Exploring Single-Cell Data with Deep Multitasking Neural Networks.”</span> Journal Article. <em>Nat Methods</em> 16 (11): 1139–45. <a href="https://doi.org/10.1038/s41592-019-0576-7">https://doi.org/10.1038/s41592-019-0576-7</a>.
</div>
<div class="csl-entry">
Arisdakessian, C., O. Poirion, B. Yunits, X. Zhu, and L. X. Garmire. 2019. <span>“DeepImpute: An Accurate, Fast, and Scalable Deep Neural Network Method to Impute Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 211. <a href="https://doi.org/10.1186/s13059-019-1837-6">https://doi.org/10.1186/s13059-019-1837-6</a>.
</div>
<div class="csl-entry">
Becht, E., L. McInnes, J. Healy, C. A. Dutertre, I. W. H. Kwok, L. G. Ng, F. Ginhoux, and E. W. Newell. 2018. <span>“Dimensionality Reduction for Visualizing Single-Cell Data Using UMAP.”</span> Journal Article. <em>Nat Biotechnol</em>. <a href="https://doi.org/10.1038/nbt.4314">https://doi.org/10.1038/nbt.4314</a>.
</div>
<div class="csl-entry">
Bost, P., A. Giladi, Y. Liu, Y. Bendjelal, G. Xu, E. David, R. Blecher-Gonen, et al. 2020. <span>“Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients.”</span> Journal Article. <em>Cell</em> 181 (7): 1475–1488 e12. <a href="https://doi.org/10.1016/j.cell.2020.05.006">https://doi.org/10.1016/j.cell.2020.05.006</a>.
</div>
<div class="csl-entry">
Butler, A., P. Hoffman, P. Smibert, E. Papalexi, and R. Satija. 2018. <span>“Integrating Single-Cell Transcriptomic Data Across Different Conditions, Technologies, and Species.”</span> Journal Article. <em>Nat Biotechnol</em> 36 (5): 411–20. <a href="https://doi.org/10.1038/nbt.4096">https://doi.org/10.1038/nbt.4096</a>.
</div>
<div class="csl-entry">
Chen, G., B. Ning, and T. Shi. 2019. <span>“Single-Cell RNA-Seq Technologies and Related Computational Data Analysis.”</span> Journal Article. <em>Front Genet</em> 10: 317. <a href="https://doi.org/10.3389/fgene.2019.00317">https://doi.org/10.3389/fgene.2019.00317</a>.
</div>
<div class="csl-entry">
Eisenstein, M. 2020. <span>“Single-Cell RNA-Seq Analysis Software Providers Scramble to Offer Solutions.”</span> Journal Article. <em>Nat Biotechnol</em> 38 (3): 254–57. <a href="https://doi.org/10.1038/s41587-020-0449-8">https://doi.org/10.1038/s41587-020-0449-8</a>.
</div>
<div class="csl-entry">
Eraslan, G., L. M. Simon, M. Mircea, N. S. Mueller, and F. J. Theis. 2019. <span>“Single-Cell RNA-Seq Denoising Using a Deep Count Autoencoder.”</span> Journal Article. <em>Nat Commun</em> 10 (1): 390. <a href="https://doi.org/10.1038/s41467-018-07931-2">https://doi.org/10.1038/s41467-018-07931-2</a>.
</div>
<div class="csl-entry">
Finak, G., A. McDavid, M. Yajima, J. Deng, V. Gersuk, A. K. Shalek, C. K. Slichter, et al. 2015. <span>“MAST: A Flexible Statistical Framework for Assessing Transcriptional Changes and Characterizing Heterogeneity in Single-Cell RNA Sequencing Data.”</span> Journal Article. <em>Genome Biol</em> 16: 278. <a href="https://doi.org/10.1186/s13059-015-0844-5">https://doi.org/10.1186/s13059-015-0844-5</a>.
</div>
<div class="csl-entry">
Hafemeister, C., and R. Satija. 2019. <span>“Normalization and Variance Stabilization of Single-Cell RNA-Seq Data Using Regularized Negative Binomial Regression.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 296. <a href="https://doi.org/10.1186/s13059-019-1874-1">https://doi.org/10.1186/s13059-019-1874-1</a>.
</div>
<div class="csl-entry">
Haghverdi, L., A. T. L. Lun, M. D. Morgan, and J. C. Marioni. 2018. <span>“Batch Effects in Single-Cell RNA-Sequencing Data Are Corrected by Matching Mutual Nearest Neighbors.”</span> Journal Article. <em>Nat Biotechnol</em> 36 (5): 421–27. <a href="https://doi.org/10.1038/nbt.4091">https://doi.org/10.1038/nbt.4091</a>.
</div>
<div class="csl-entry">
Huang, M., J. Wang, E. Torre, H. Dueck, S. Shaffer, R. Bonasio, J. I. Murray, A. Raj, M. Li, and N. R. Zhang. 2018. <span>“SAVER: Gene Expression Recovery for Single-Cell RNA Sequencing.”</span> Journal Article. <em>Nat Methods</em> 15 (7): 539–42. <a href="https://doi.org/10.1038/s41592-018-0033-z">https://doi.org/10.1038/s41592-018-0033-z</a>.
</div>
<div class="csl-entry">
Kharchenko, P. V., L. Silberstein, and D. T. Scadden. 2014. <span>“Bayesian Approach to Single-Cell Differential Expression Analysis.”</span> Journal Article. <em>Nat Methods</em> 11 (7): 740–42. <a href="https://doi.org/10.1038/nmeth.2967">https://doi.org/10.1038/nmeth.2967</a>.
</div>
<div class="csl-entry">
Kinker, G. S., A. C. Greenwald, R. Tal, Z. Orlova, M. S. Cuoco, J. M. McFarland, A. Warren, et al. 2020. <span>“Pan-Cancer Single-Cell RNA-Seq Identifies Recurring Programs of Cellular Heterogeneity.”</span> Journal Article. <em>Nat Genet</em> 52 (11): 1208–18. <a href="https://doi.org/10.1038/s41588-020-00726-6">https://doi.org/10.1038/s41588-020-00726-6</a>.
</div>
<div class="csl-entry">
Korsunsky, I., N. Millard, J. Fan, K. Slowikowski, F. Zhang, K. Wei, Y. Baglaenko, M. Brenner, P. R. Loh, and S. Raychaudhuri. 2019. <span>“Fast, Sensitive and Accurate Integration of Single-Cell Data with Harmony.”</span> Journal Article. <em>Nat Methods</em> 16 (12): 1289–96. <a href="https://doi.org/10.1038/s41592-019-0619-0">https://doi.org/10.1038/s41592-019-0619-0</a>.
</div>
<div class="csl-entry">
Lahnemann, D., J. Koster, E. Szczurek, D. J. McCarthy, S. C. Hicks, M. D. Robinson, C. A. Vallejos, et al. 2020. <span>“Eleven Grand Challenges in Single-Cell Data Science.”</span> Journal Article. <em>Genome Biol</em> 21 (1): 31. <a href="https://doi.org/10.1186/s13059-020-1926-6">https://doi.org/10.1186/s13059-020-1926-6</a>.
</div>
<div class="csl-entry">
Li, W. V., and J. J. Li. 2018. <span>“An Accurate and Robust Imputation Method scImpute for Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Nat Commun</em> 9 (1): 997. <a href="https://doi.org/10.1038/s41467-018-03405-7">https://doi.org/10.1038/s41467-018-03405-7</a>.
</div>
<div class="csl-entry">
Linderman, G. C., M. Rachh, J. G. Hoskins, S. Steinerberger, and Y. Kluger. 2019. <span>“Fast Interpolation-Based t-SNE for Improved Visualization of Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Nat Methods</em> 16 (3): 243–45. <a href="https://doi.org/10.1038/s41592-018-0308-4">https://doi.org/10.1038/s41592-018-0308-4</a>.
</div>
<div class="csl-entry">
Lopez, R., J. Regier, M. B. Cole, M. I. Jordan, and N. Yosef. 2018. <span>“Deep Generative Modeling for Single-Cell Transcriptomics.”</span> Journal Article. <em>Nat Methods</em> 15 (12): 1053–58. <a href="https://doi.org/10.1038/s41592-018-0229-2">https://doi.org/10.1038/s41592-018-0229-2</a>.
</div>
<div class="csl-entry">
Lun, A. T., K. Bach, and J. C. Marioni. 2016. <span>“Pooling Across Cells to Normalize Single-Cell RNA Sequencing Data with Many Zero Counts.”</span> Journal Article. <em>Genome Biol</em> 17: 75. <a href="https://doi.org/10.1186/s13059-016-0947-7">https://doi.org/10.1186/s13059-016-0947-7</a>.
</div>
<div class="csl-entry">
Maaten, G. van der, L. &amp; Hinton. 2008. <span>“Visualizing Data Using t-SNE.”</span> Journal Article. <em>J. Mach. Learn</em> 9: 2579–2605.
</div>
<div class="csl-entry">
Macosko, E. Z., A. Basu, R. Satija, J. Nemesh, K. Shekhar, M. Goldman, I. Tirosh, et al. 2015. <span>“Highly Parallel Genome-Wide Expression Profiling of Individual Cells Using Nanoliter Droplets.”</span> Journal Article. <em>Cell</em> 161 (5): 1202–14. <a href="https://doi.org/10.1016/j.cell.2015.05.002">https://doi.org/10.1016/j.cell.2015.05.002</a>.
</div>
<div class="csl-entry">
Mannarapu, M., B. Dariya, and O. R. Bandapalli. 2021. <span>“Application of Single-Cell Sequencing Technologies in Pancreatic Cancer.”</span> Journal Article. <em>Mol Cell Biochem</em> 476 (6): 2429–37. <a href="https://doi.org/10.1007/s11010-021-04095-4">https://doi.org/10.1007/s11010-021-04095-4</a>.
</div>
<div class="csl-entry">
Miao, Z., K. Deng, X. Wang, and X. Zhang. 2018. <span>“DEsingle for Detecting Three Types of Differential Expression in Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Bioinformatics</em> 34 (18): 3223–24. <a href="https://doi.org/10.1093/bioinformatics/bty332">https://doi.org/10.1093/bioinformatics/bty332</a>.
</div>
<div class="csl-entry">
Navin, N. E. 2015. <span>“The First Five Years of Single-Cell Cancer Genomics and Beyond.”</span> Journal Article. <em>Genome Res</em> 25 (10): 1499–1507. <a href="https://doi.org/10.1101/gr.191098.115">https://doi.org/10.1101/gr.191098.115</a>.
</div>
<div class="csl-entry">
Peng, T., Q. Zhu, P. Yin, and K. Tan. 2019. <span>“SCRABBLE: Single-Cell RNA-Seq Imputation Constrained by Bulk RNA-Seq Data.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 88. <a href="https://doi.org/10.1186/s13059-019-1681-8">https://doi.org/10.1186/s13059-019-1681-8</a>.
</div>
<div class="csl-entry">
Petegrosso, R., Z. Li, and R. Kuang. 2020. <span>“Machine Learning and Statistical Methods for Clustering Single-Cell RNA-Sequencing Data.”</span> Journal Article. <em>Brief Bioinform</em> 21 (4): 1209–23. <a href="https://doi.org/10.1093/bib/bbz063">https://doi.org/10.1093/bib/bbz063</a>.
</div>
<div class="csl-entry">
Picelli, S., A. K. Bjorklund, O. R. Faridani, S. Sagasser, G. Winberg, and R. Sandberg. 2013. <span>“Smart-Seq2 for Sensitive Full-Length Transcriptome Profiling in Single Cells.”</span> Journal Article. <em>Nat Methods</em> 10 (11): 1096–98. <a href="https://doi.org/10.1038/nmeth.2639">https://doi.org/10.1038/nmeth.2639</a>.
</div>
<div class="csl-entry">
Roweis, S. T., and L. K. Saul. 2000. <span>“Nonlinear Dimensionality Reduction by Locally Linear Embedding.”</span> Journal Article. <em>Science</em> 290 (5500): 2323–26. <a href="https://doi.org/10.1126/science.290.5500.2323">https://doi.org/10.1126/science.290.5500.2323</a>.
</div>
<div class="csl-entry">
Srinivasan, S., A. Leshchyk, N. T. Johnson, and D. Korkin. 2020. <span>“A Hybrid Deep Clustering Approach for Robust Cell Type Profiling Using Single-Cell RNA-Seq Data.”</span> Journal Article. <em>RNA</em> 26 (10): 1303–19. <a href="https://doi.org/10.1261/rna.074427.119">https://doi.org/10.1261/rna.074427.119</a>.
</div>
<div class="csl-entry">
Stuart, T., A. Butler, P. Hoffman, C. Hafemeister, E. Papalexi, 3rd Mauck W. M., Y. Hao, M. Stoeckius, P. Smibert, and R. Satija. 2019. <span>“Comprehensive Integration of Single-Cell Data.”</span> Journal Article. <em>Cell</em> 177 (7): 1888–1902 e21. <a href="https://doi.org/10.1016/j.cell.2019.05.031">https://doi.org/10.1016/j.cell.2019.05.031</a>.
</div>
<div class="csl-entry">
Subelj, L., and M. Bajec. 2011. <span>“Unfolding Communities in Large Complex Networks: Combining Defensive and Offensive Label Propagation for Core Extraction.”</span> Journal Article. <em>Phys Rev E Stat Nonlin Soft Matter Phys</em> 83 (3 Pt 2): 036103. <a href="https://doi.org/10.1103/PhysRevE.83.036103">https://doi.org/10.1103/PhysRevE.83.036103</a>.
</div>
<div class="csl-entry">
Traag, V. A., L. Waltman, and N. J. van Eck. 2019. <span>“From Louvain to Leiden: Guaranteeing Well-Connected Communities.”</span> Journal Article. <em>Sci Rep</em> 9 (1): 5233. <a href="https://doi.org/10.1038/s41598-019-41695-z">https://doi.org/10.1038/s41598-019-41695-z</a>.
</div>
<div class="csl-entry">
Tran, H. T. N., K. S. Ang, M. Chevrier, X. Zhang, N. Y. S. Lee, M. Goh, and J. Chen. 2020. <span>“A Benchmark of Batch-Effect Correction Methods for Single-Cell RNA Sequencing Data.”</span> Journal Article. <em>Genome Biol</em> 21 (1): 12. <a href="https://doi.org/10.1186/s13059-019-1850-9">https://doi.org/10.1186/s13059-019-1850-9</a>.
</div>
<div class="csl-entry">
Vallejos, C. A., J. C. Marioni, and S. Richardson. 2015. <span>“BASiCS: Bayesian Analysis of Single-Cell Sequencing Data.”</span> Journal Article. <em>PLoS Comput Biol</em> 11 (6): e1004333. <a href="https://doi.org/10.1371/journal.pcbi.1004333">https://doi.org/10.1371/journal.pcbi.1004333</a>.
</div>
<div class="csl-entry">
Vitak, S. A., K. A. Torkenczy, J. L. Rosenkrantz, A. J. Fields, L. Christiansen, M. H. Wong, L. Carbone, F. J. Steemers, and A. Adey. 2017. <span>“Sequencing Thousands of Single-Cell Genomes with Combinatorial Indexing.”</span> Journal Article. <em>Nat Methods</em> 14 (3): 302–8. <a href="https://doi.org/10.1038/nmeth.4154">https://doi.org/10.1038/nmeth.4154</a>.
</div>
<div class="csl-entry">
Wang, B., J. Zhu, E. Pierson, D. Ramazzotti, and S. Batzoglou. 2017. <span>“Visualization and Analysis of Single-Cell RNA-Seq Data by Kernel-Based Similarity Learning.”</span> Journal Article. <em>Nat Methods</em> 14 (4): 414–16. <a href="https://doi.org/10.1038/nmeth.4207">https://doi.org/10.1038/nmeth.4207</a>.
</div>
<div class="csl-entry">
Wauters, E., P. Van Mol, A. D. Garg, S. Jansen, Y. Van Herck, L. Vanderbeke, A. Bassez, et al. 2021. <span>“Discriminating Mild from Critical COVID-19 by Innate and Adaptive Immune Single-Cell Profiling of Bronchoalveolar Lavages.”</span> Journal Article. <em>Cell Res</em> 31 (3): 272–90. <a href="https://doi.org/10.1038/s41422-020-00455-9">https://doi.org/10.1038/s41422-020-00455-9</a>.
</div>
<div class="csl-entry">
Welch, J. D., A. J. Hartemink, and J. F. Prins. 2016. <span>“SLICER: Inferring Branched, Nonlinear Cellular Trajectories from Single Cell RNA-Seq Data.”</span> Journal Article. <em>Genome Biol</em> 17 (1): 106. <a href="https://doi.org/10.1186/s13059-016-0975-3">https://doi.org/10.1186/s13059-016-0975-3</a>.
</div>
<div class="csl-entry">
Wolf, F. A., P. Angerer, and F. J. Theis. 2018. <span>“SCANPY: Large-Scale Single-Cell Gene Expression Data Analysis.”</span> Journal Article. <em>Genome Biol</em> 19 (1): 15. <a href="https://doi.org/10.1186/s13059-017-1382-0">https://doi.org/10.1186/s13059-017-1382-0</a>.
</div>
<div class="csl-entry">
Wolock, S. L., R. Lopez, and A. M. Klein. 2019. <span>“Scrublet: Computational Identification of Cell Doublets in Single-Cell Transcriptomic Data.”</span> Journal Article. <em>Cell Syst</em> 8 (4): 281–291 e9. <a href="https://doi.org/10.1016/j.cels.2018.11.005">https://doi.org/10.1016/j.cels.2018.11.005</a>.
</div>
<div class="csl-entry">
Xu, Y., Z. Zhang, L. You, J. Liu, Z. Fan, and X. Zhou. 2020. <span>“scIGANs: Single-Cell RNA-Seq Imputation Using Generative Adversarial Networks.”</span> Journal Article. <em>Nucleic Acids Res</em> 48 (15): e85. <a href="https://doi.org/10.1093/nar/gkaa506">https://doi.org/10.1093/nar/gkaa506</a>.
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ch-6.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/Huang-AI4Medicine-Lab/survey-of-DL-for-scRNA-seq-analysis/edit/main/07-Tables.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["survey-of-DL-for-scRNA-seq-analysis.pdf", "survey-of-DL-for-scRNA-seq-analysis.epub"],
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
