<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis</title>
  <meta name="description" content="7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  <meta name="generator" content="bookdown 0.23 and GitBook 2.6.7" />

  <meta property="og:title" content="7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  <meta property="og:type" content="book" />
  
  
  
  <meta name="github-repo" content="Huang-AI4Medicine-Lab/survey-of-DL-for-scRNA-seq-analysis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7 Tables | Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis" />
  
  
  

<meta name="author" content="Mario Flores1§, Zhentao Liu1, Tinghe Zhang1, Md Musaddaqui Hasib1, Yu-Chiao Chiu2, Zhenqing Ye2,3, Karla Paniagua1, Sumin Jo1, Jianqiu Zhang1, Shou-Jiang Gao4,6, Yufang Jin1, Yidong Chen2,3§, and Yufei Huang5,6§" />


<meta name="date" content="2021-09-23" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="tables.html"/>

<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet" />
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Survey of Deep Learning for scRNA-seq Analysis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About the authors</a></li>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html"><i class="fa fa-check"></i>About this book</a>
<ul>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="about-this-book.html"><a href="about-this-book.html#key-points"><i class="fa fa-check"></i>Key Points</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="ch-1.html"><a href="ch-1.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="ch-2.html"><a href="ch-2.html"><i class="fa fa-check"></i><b>2</b> Overview of scRNA-seq processing pipeline</a></li>
<li class="chapter" data-level="3" data-path="ch-3.html"><a href="ch-3.html"><i class="fa fa-check"></i><b>3</b> Overview of common deep learning models for scRNA-seq analysis</a>
<ul>
<li class="chapter" data-level="3.1" data-path="ch-3.html"><a href="ch-3.html#ch-3-1"><i class="fa fa-check"></i><b>3.1</b> Variational Autoencoder (VAEs) for scRNA-Seq data</a></li>
<li class="chapter" data-level="3.2" data-path="ch-3.html"><a href="ch-3.html#ch-3-2"><i class="fa fa-check"></i><b>3.2</b> Autoencoders (AEs) for scRNA-seq data</a></li>
<li class="chapter" data-level="3.3" data-path="ch-3.html"><a href="ch-3.html#ch-3-3"><i class="fa fa-check"></i><b>3.3</b> Generative adversarial networks (GANs) for scRNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-4.html"><a href="ch-4.html"><i class="fa fa-check"></i><b>4</b> Overview of datasets and evaluation metrics</a>
<ul>
<li class="chapter" data-level="4.1" data-path="ch-4.html"><a href="ch-4.html#ch-4-1"><i class="fa fa-check"></i><b>4.1</b> Evaluation methods</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-1"><i class="fa fa-check"></i><b>4.1.1</b> Imputation</a></li>
<li class="chapter" data-level="4.1.2" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-2"><i class="fa fa-check"></i><b>4.1.2</b> Batch effect correction</a></li>
<li class="chapter" data-level="4.1.3" data-path="ch-4.html"><a href="ch-4.html#ch-4-1-3"><i class="fa fa-check"></i><b>4.1.3</b> Clustering</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch-5.html"><a href="ch-5.html"><i class="fa fa-check"></i><b>5</b> Survey of deep learning models for scRNA-Seq analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-1"><i class="fa fa-check"></i><b>5.1</b> Imputation</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-1"><i class="fa fa-check"></i><b>5.1.1</b> DCA:deep count autoencoder</a></li>
<li class="chapter" data-level="5.1.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-2"><i class="fa fa-check"></i><b>5.1.2</b> SAVER-X: single-cell analysis via expression recovery harnessing external data</a></li>
<li class="chapter" data-level="5.1.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-3"><i class="fa fa-check"></i><b>5.1.3</b> DeepImpute (Deep neural network Imputation)</a></li>
<li class="chapter" data-level="5.1.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-4"><i class="fa fa-check"></i><b>5.1.4</b> LATE: Learning with AuToEncoder</a></li>
<li class="chapter" data-level="5.1.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-5"><i class="fa fa-check"></i><b>5.1.5</b> scGMAI</a></li>
<li class="chapter" data-level="5.1.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-1-6"><i class="fa fa-check"></i><b>5.1.6</b> scIGANs</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-2"><i class="fa fa-check"></i><b>5.2</b> Batch effect correction</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-1"><i class="fa fa-check"></i><b>5.2.1</b> BERMUDA: Batch Effect ReMoval Using Deep Autoencoders</a></li>
<li class="chapter" data-level="5.2.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-2"><i class="fa fa-check"></i><b>5.2.2</b> DESC: batch correction based on clustering</a></li>
<li class="chapter" data-level="5.2.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-2-3"><i class="fa fa-check"></i><b>5.2.3</b> iMAP: Integration of Multiple single-cell datasets by Adversarial Paired-style transfer networks</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-3"><i class="fa fa-check"></i><b>5.3</b> Dimension reduction, latent representation, clustering, and data augmentation</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-1"><i class="fa fa-check"></i><b>5.3.1</b> Dimension reduction by AEs with gene-interaction constrained architecture</a></li>
<li class="chapter" data-level="5.3.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-2"><i class="fa fa-check"></i><b>5.3.2</b> Dhaka: a VAE-based dimension reduction model</a></li>
<li class="chapter" data-level="5.3.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-3"><i class="fa fa-check"></i><b>5.3.3</b> cvis: a VAE for capturing low-dimensional structures</a></li>
<li class="chapter" data-level="5.3.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-4"><i class="fa fa-check"></i><b>5.3.4</b> scVAE: VAE for single-cell gene expression data</a></li>
<li class="chapter" data-level="5.3.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-5"><i class="fa fa-check"></i><b>5.3.5</b> VASC: VAE for scRNA-seq</a></li>
<li class="chapter" data-level="5.3.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-6"><i class="fa fa-check"></i><b>5.3.6</b> scDeepCluster</a></li>
<li class="chapter" data-level="5.3.7" data-path="ch-5.html"><a href="ch-5.html#ch-5-3-7"><i class="fa fa-check"></i><b>5.3.7</b> cscGAN: Conditional single-cell generative adversarial neural networks</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-4"><i class="fa fa-check"></i><b>5.4</b> Multi-functional models</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-1"><i class="fa fa-check"></i><b>5.4.1</b> scVI: single-cell variational inference</a></li>
<li class="chapter" data-level="5.4.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-2"><i class="fa fa-check"></i><b>5.4.2</b> LDVAE: linearly decoded variational autoencoder</a></li>
<li class="chapter" data-level="5.4.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-3"><i class="fa fa-check"></i><b>5.4.3</b> SAUCIE</a></li>
<li class="chapter" data-level="5.4.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-4-4"><i class="fa fa-check"></i><b>5.4.4</b> scScope</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="ch-5.html"><a href="ch-5.html#ch-5-5"><i class="fa fa-check"></i><b>5.5</b> Doublet Classification</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-5-1"><i class="fa fa-check"></i><b>5.5.1</b> Solo</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="ch-5.html"><a href="ch-5.html#ch-5-6"><i class="fa fa-check"></i><b>5.6</b> Automated cell type identification</a>
<ul>
<li class="chapter" data-level="5.6.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-1"><i class="fa fa-check"></i><b>5.6.1</b> DigitalDLSorter</a></li>
<li class="chapter" data-level="5.6.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-2"><i class="fa fa-check"></i><b>5.6.2</b> scCapsNet</a></li>
<li class="chapter" data-level="5.6.3" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-3"><i class="fa fa-check"></i><b>5.6.3</b> netAE: network-enhanced autoencoder</a></li>
<li class="chapter" data-level="5.6.4" data-path="ch-5.html"><a href="ch-5.html#ch-5-6-4"><i class="fa fa-check"></i><b>5.6.4</b> scDGN - supervised adversarial alignment of single-cell RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="ch-5.html"><a href="ch-5.html#ch-5-7"><i class="fa fa-check"></i><b>5.7</b> Biological function prediction</a>
<ul>
<li class="chapter" data-level="5.7.1" data-path="ch-5.html"><a href="ch-5.html#ch-5-7-1"><i class="fa fa-check"></i><b>5.7.1</b> CNNC: convolutional neural network for coexpression</a></li>
<li class="chapter" data-level="5.7.2" data-path="ch-5.html"><a href="ch-5.html#ch-5-7-2"><i class="fa fa-check"></i><b>5.7.2</b> scGen, a generative model to predict perturbation response of single cells across cell types</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="ch-6.html"><a href="ch-6.html"><i class="fa fa-check"></i><b>6</b> Conclusion and Discussions</a></li>
<li class="chapter" data-level="" data-path="tables.html"><a href="tables.html"><i class="fa fa-check"></i>Tables</a>
<ul>
<li class="chapter" data-level="6.1" data-path="tables.html"><a href="tables.html#table1"><i class="fa fa-check"></i><b>6.1</b> Table1</a></li>
<li class="chapter" data-level="6.2" data-path="tables.html"><a href="tables.html#table2"><i class="fa fa-check"></i><b>6.2</b> Table2</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="tables.html"><a href="tables.html#table2a"><i class="fa fa-check"></i><b>6.2.1</b> Table2a</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="ch-8.html"><a href="ch-8.html"><i class="fa fa-check"></i><b>7</b> Tables</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Deep learning tackles single-cell analysis - A survey of deep learning for scRNA-seq analysis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch-8" class="section level1" number="7">
<h1><span class="header-section-number">7</span> Tables</h1>
<p>Embedded table test, Table <a href="ch-8.html#tab:Table11">7.1</a></p>
<div style="border: 1px solid #ddd; padding: 5px; overflow-x: scroll; width:100%; ">
<table class="table table-striped" style="margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption>
<span id="tab:Table11">Table 7.1: </span>Deep Learning algorithms reviewed in the paper
</caption>
<thead>
<tr>
<th style="text-align:left;">
App
</th>
<th style="text-align:left;">
Algorithm
</th>
<th style="text-align:left;">
Models
</th>
<th style="text-align:left;">
Evaluation
</th>
<th style="text-align:left;">
Environment
</th>
<th style="text-align:left;">
Codes
</th>
<th style="text-align:left;">
Refs
</th>
</tr>
</thead>
<tbody>
<tr grouplength="6">
<td colspan="7" style="background-color: #666; color: #fff;">
<strong>Imputation</strong>
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
DCA
</td>
<td style="text-align:left;">
AE
</td>
<td style="text-align:left;">
DREMI
</td>
<td style="text-align:left;">
Keras, Tensorflow, scanpy
</td>
<td style="text-align:left;">
<a href="https://github.com/theislab/dca" class="uri">https://github.com/theislab/dca</a>
</td>
<td style="text-align:left;">
<span class="citation"><a href="#ref-RN100" role="doc-biblioref">Wolf, Angerer, and Theis</a> (<a href="#ref-RN100" role="doc-biblioref">2018</a>)</span>
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
SAVER-X
</td>
<td style="text-align:left;">
AE+TL
</td>
<td style="text-align:left;">
t-SNE, ARI
</td>
<td style="text-align:left;">
R/sctransfer
</td>
<td style="text-align:left;">
<a href="https://github.com/jingshuw/SAVERX" class="uri">https://github.com/jingshuw/SAVERX</a>
</td>
<td style="text-align:left;">
[51]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
DeepImpute
</td>
<td style="text-align:left;">
DNN
</td>
<td style="text-align:left;">
MSE, Pearson’s correlation
</td>
<td style="text-align:left;">
Keras/Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/lanagarmire/DeepImpute" class="uri">https://github.com/lanagarmire/DeepImpute</a>
</td>
<td style="text-align:left;">
[19]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
LATE
</td>
<td style="text-align:left;">
AE
</td>
<td style="text-align:left;">
MSE
</td>
<td style="text-align:left;">
Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/audreyqyfu/LATE" class="uri">https://github.com/audreyqyfu/LATE</a>
</td>
<td style="text-align:left;">
[52]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scGAMI
</td>
<td style="text-align:left;">
AE
</td>
<td style="text-align:left;">
NMI, ARI, HS and CS
</td>
<td style="text-align:left;">
Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/QUST-AIBBDRC/scGMAI/" class="uri">https://github.com/QUST-AIBBDRC/scGMAI/</a>
</td>
<td style="text-align:left;">
[53]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scIGANs
</td>
<td style="text-align:left;">
GAN
</td>
<td style="text-align:left;">
ARI, ACC, AUC, and F-score
</td>
<td style="text-align:left;">
PyTorch
</td>
<td style="text-align:left;">
<a href="https://github.com/xuyungang/scIGANs" class="uri">https://github.com/xuyungang/scIGANs</a>
</td>
<td style="text-align:left;">
[18]
</td>
</tr>
<tr grouplength="3">
<td colspan="7" style="background-color: #666; color: #fff;">
<strong>Batch correction</strong>
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
BERMUDA
</td>
<td style="text-align:left;">
AE+TL
</td>
<td style="text-align:left;">
kBET, entropy of Mixing, SI
</td>
<td style="text-align:left;">
PyTorch
</td>
<td style="text-align:left;">
<a href="https://github.com/txWang/BERMUDA" class="uri">https://github.com/txWang/BERMUDA</a>
</td>
<td style="text-align:left;">
[57]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
DESC
</td>
<td style="text-align:left;">
AE
</td>
<td style="text-align:left;">
ARI, KL
</td>
<td style="text-align:left;">
Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/eleozzr/desc" class="uri">https://github.com/eleozzr/desc</a>
</td>
<td style="text-align:left;">
[61]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
iMAP
</td>
<td style="text-align:left;">
AE+GAN
</td>
<td style="text-align:left;">
kBET, LISI
</td>
<td style="text-align:left;">
PyTorch
</td>
<td style="text-align:left;">
<a href="https://github.com/Svvord/iMAP" class="uri">https://github.com/Svvord/iMAP</a>
</td>
<td style="text-align:left;">
[64]
</td>
</tr>
<tr grouplength="6">
<td colspan="7" style="background-color: #666; color: #fff;">
<strong>Clustering, latent representation, dimension reduction, and data augmentation</strong>
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
Dhaka
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
ARI, Spearman Correlation
</td>
<td style="text-align:left;">
Keras/Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/MicrosoftGenomics/Dhaka" class="uri">https://github.com/MicrosoftGenomics/Dhaka</a>
</td>
<td style="text-align:left;">
[66]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scvis
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
KNN preservation, log-likelihood
</td>
<td style="text-align:left;">
Tensorflow
</td>
<td style="text-align:left;">
<a href="https://bitbucket.org/jerry00/scvis-dev/src/master/" class="uri">https://bitbucket.org/jerry00/scvis-dev/src/master/</a>
</td>
<td style="text-align:left;">
[69]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scVAE
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
ARI
</td>
<td style="text-align:left;">
Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/scvae/scvae" class="uri">https://github.com/scvae/scvae</a>
</td>
<td style="text-align:left;">
[70]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
VASC
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
NMI, ARI, HS, and CS
</td>
<td style="text-align:left;">
H5py, Keras
</td>
<td style="text-align:left;">
<a href="https://github.com/wang-research/VASC" class="uri">https://github.com/wang-research/VASC</a>
</td>
<td style="text-align:left;">
[71]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scDeepCluster
</td>
<td style="text-align:left;">
AE
</td>
<td style="text-align:left;">
ARI, NMI, clustering accuracy
</td>
<td style="text-align:left;">
Keras, Scanpy
</td>
<td style="text-align:left;">
<a href="https://github.com/ttgump/scDeepCluster" class="uri">https://github.com/ttgump/scDeepCluster</a>
</td>
<td style="text-align:left;">
[73]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
cscGAN
</td>
<td style="text-align:left;">
GAN
</td>
<td style="text-align:left;">
t-SNE, marker genes, MMD, AUC
</td>
<td style="text-align:left;">
Scipy, Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/imsb-uke/scGAN" class="uri">https://github.com/imsb-uke/scGAN</a>
</td>
<td style="text-align:left;">
[76]
</td>
</tr>
<tr grouplength="4">
<td colspan="7" style="background-color: #666; color: #fff;">
<strong>Multi-functional models (IM: imputation, BC: batch correction, CL: clustering)</strong>
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scVI
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
IM: L1 distance; CL: ARI, NMI, SI; BC: Entropy of Mixing
</td>
<td style="text-align:left;">
PyTorch, Anndata
</td>
<td style="text-align:left;">
<a href="https://github.com/YosefLab/scvi-tools" class="uri">https://github.com/YosefLab/scvi-tools</a>
</td>
<td style="text-align:left;">
[16]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
LDVAE
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
Reconstruction errors
</td>
<td style="text-align:left;">
Part of scVI
</td>
<td style="text-align:left;">
<a href="https://github.com/YosefLab/scvi-tools" class="uri">https://github.com/YosefLab/scvi-tools</a>
</td>
<td style="text-align:left;">
[80]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
SAUCIE
</td>
<td style="text-align:left;">
AE
</td>
<td style="text-align:left;">
IM: R2 statistics; CL: SI; BC: modified kBET; Visualization: Precision/Recall
</td>
<td style="text-align:left;">
Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/KrishnaswamyLab/SAUCIE/" class="uri">https://github.com/KrishnaswamyLab/SAUCIE/</a>
</td>
<td style="text-align:left;">
[14]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scScope
</td>
<td style="text-align:left;">
AE
</td>
<td style="text-align:left;">
<a href="IM:Reconstruction" class="uri">IM:Reconstruction</a> errors; BC: Entropy of mixing; CL: ARI
</td>
<td style="text-align:left;">
Tensorflow, Scikit-learn
</td>
<td style="text-align:left;">
<a href="https://github.com/AltschulerWu-Lab/scScope" class="uri">https://github.com/AltschulerWu-Lab/scScope</a>
</td>
<td style="text-align:left;">
[86]
</td>
</tr>
<tr grouplength="4">
<td colspan="7" style="background-color: #666; color: #fff;">
<strong>Cell type Identification</strong>
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
DigitalDLSorter
</td>
<td style="text-align:left;">
DNN
</td>
<td style="text-align:left;">
Pearson correlation
</td>
<td style="text-align:left;">
R/Python/Keras
</td>
<td style="text-align:left;">
<a href="https://github.com/cartof/digitalDLSorter" class="uri">https://github.com/cartof/digitalDLSorter</a>
</td>
<td style="text-align:left;">
[87]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scCapsNet
</td>
<td style="text-align:left;">
CapsNet
</td>
<td style="text-align:left;">
Cell-type Prediction accuracy
</td>
<td style="text-align:left;">
Keras, Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/wanglf19/scCaps" class="uri">https://github.com/wanglf19/scCaps</a>
</td>
<td style="text-align:left;">
[94]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
netAE
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
Cell-type Prediction accuracy, t-SNE for visualization
</td>
<td style="text-align:left;">
PyTorch
</td>
<td style="text-align:left;">
<a href="https://github.com/LeoZDong/netAE" class="uri">https://github.com/LeoZDong/netAE</a>
</td>
<td style="text-align:left;">
[98]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scDGN
</td>
<td style="text-align:left;">
DANN
</td>
<td style="text-align:left;">
Prediciton accuracy
</td>
<td style="text-align:left;">
PyTorch
</td>
<td style="text-align:left;">
<a href="https://github.com/SongweiGe/scDGN" class="uri">https://github.com/SongweiGe/scDGN</a>
</td>
<td style="text-align:left;">
[101]
</td>
</tr>
<tr grouplength="2">
<td colspan="7" style="background-color: #666; color: #fff;">
<strong>Function analysis</strong>
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
CNNC
</td>
<td style="text-align:left;">
CNN
</td>
<td style="text-align:left;">
AUROC, AUPRC, and accuracy
</td>
<td style="text-align:left;">
Keras, Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/xiaoyeye/CNNC" class="uri">https://github.com/xiaoyeye/CNNC</a>
</td>
<td style="text-align:left;">
[104]
</td>
</tr>
<tr>
<td style="text-align:left;padding-left: 2em;width: 10em; " indentlevel="1">
</td>
<td style="text-align:left;">
scGen
</td>
<td style="text-align:left;">
VAE
</td>
<td style="text-align:left;">
Correlation, visualization
</td>
<td style="text-align:left;">
Tensorflow
</td>
<td style="text-align:left;">
<a href="https://github.com/theislab/scgen" class="uri">https://github.com/theislab/scgen</a>
</td>
<td style="text-align:left;">
[113]
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<sup></sup> DL Model keywords: AE: autoencoder, AE+TL: autoencoder with transfer learning, AE: variational autoencoder, GAN: Generative adversarial network, CNN: convolutional neural network, DNN: deep neural network, DANN: domain adversarial neural network, CapsNet: capsule neural network
</td>
</tr>
</tfoot>
</table>
</div>

<div id="refs" class="references csl-bib-body hanging-indent">
<div class="csl-entry">
Abdelaal, T., L. Michielsen, D. Cats, D. Hoogduin, H. Mei, M. J. T. Reinders, and A. Mahfouz. 2019. <span>“A Comparison of Automatic Cell Identification Methods for Single-Cell RNA Sequencing Data.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 194. <a href="https://doi.org/10.1186/s13059-019-1795-z">https://doi.org/10.1186/s13059-019-1795-z</a>.
</div>
<div class="csl-entry">
Aibar, S., C. B. Gonzalez-Blas, T. Moerman, V. A. Huynh-Thu, H. Imrichova, G. Hulselmans, F. Rambow, et al. 2017. <span>“SCENIC: Single-Cell Regulatory Network Inference and Clustering.”</span> Journal Article. <em>Nat Methods</em> 14 (11): 1083–86. <a href="https://doi.org/10.1038/nmeth.4463">https://doi.org/10.1038/nmeth.4463</a>.
</div>
<div class="csl-entry">
Alavi, A., M. Ruffalo, A. Parvangada, Z. Huang, and Z. Bar-Joseph. 2018. <span>“A Web Server for Comparative Analysis of Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Nat Commun</em> 9 (1): 4768. <a href="https://doi.org/10.1038/s41467-018-07165-2">https://doi.org/10.1038/s41467-018-07165-2</a>.
</div>
<div class="csl-entry">
Amodio, M., D. van Dijk, K. Srinivasan, W. S. Chen, H. Mohsen, K. R. Moon, A. Campbell, et al. 2019. <span>“Exploring Single-Cell Data with Deep Multitasking Neural Networks.”</span> Journal Article. <em>Nat Methods</em> 16 (11): 1139–45. <a href="https://doi.org/10.1038/s41592-019-0576-7">https://doi.org/10.1038/s41592-019-0576-7</a>.
</div>
<div class="csl-entry">
Aran, D., Z. Hu, and A. J. Butte. 2017. <span>“xCell: Digitally Portraying the Tissue Cellular Heterogeneity Landscape.”</span> Journal Article. <em>Genome Biol</em> 18 (1): 220. <a href="https://doi.org/10.1186/s13059-017-1349-1">https://doi.org/10.1186/s13059-017-1349-1</a>.
</div>
<div class="csl-entry">
Arisdakessian, C., O. Poirion, B. Yunits, X. Zhu, and L. X. Garmire. 2019. <span>“DeepImpute: An Accurate, Fast, and Scalable Deep Neural Network Method to Impute Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 211. <a href="https://doi.org/10.1186/s13059-019-1837-6">https://doi.org/10.1186/s13059-019-1837-6</a>.
</div>
<div class="csl-entry">
Badsha, M. B., R. Li, B. Liu, Y. I. Li, M. Xian, N. E. Banovich, and A. Q. Fu. 2020. <span>“Imputation of Single-Cell Gene Expression with an Autoencoder Neural Network.”</span> Journal Article. <em>Quant Biol</em> 8 (1): 78–94. <a href="https://doi.org/10.1007/s40484-019-0192-7">https://doi.org/10.1007/s40484-019-0192-7</a>.
</div>
<div class="csl-entry">
Becht, E., N. A. Giraldo, L. Lacroix, B. Buttard, N. Elarouci, F. Petitprez, J. Selves, et al. 2016. <span>“Estimating the Population Abundance of Tissue-Infiltrating Immune and Stromal Cell Populations Using Gene Expression.”</span> Journal Article. <em>Genome Biol</em> 17 (1): 218. <a href="https://doi.org/10.1186/s13059-016-1070-5">https://doi.org/10.1186/s13059-016-1070-5</a>.
</div>
<div class="csl-entry">
Becht, E., L. McInnes, J. Healy, C. A. Dutertre, I. W. H. Kwok, L. G. Ng, F. Ginhoux, and E. W. Newell. 2018. <span>“Dimensionality Reduction for Visualizing Single-Cell Data Using UMAP.”</span> Journal Article. <em>Nat Biotechnol</em>. <a href="https://doi.org/10.1038/nbt.4314">https://doi.org/10.1038/nbt.4314</a>.
</div>
<div class="csl-entry">
Bernstein, N. J., N. L. Fong, I. Lam, M. A. Roy, D. G. Hendrickson, and D. R. Kelley. 2020. <span>“Solo: Doublet Identification in Single-Cell RNA-Seq via Semi-Supervised Deep Learning.”</span> Journal Article. <em>Cell Syst</em> 11 (1): 95–101 e5. <a href="https://doi.org/10.1016/j.cels.2020.05.010">https://doi.org/10.1016/j.cels.2020.05.010</a>.
</div>
<div class="csl-entry">
Berthelot, Schumm, D., and L. Metz. 2017. <span>“BEGAN: Boundary Equilibrium Generative Adversarial Networks.”</span> Journal Article. <em>arXiv</em>.
</div>
<div class="csl-entry">
Borgwardt, K. M., A. Gretton, M. J. Rasch, H. P. Kriegel, B. Scholkopf, and A. J. Smola. 2006. <span>“Integrating Structured Biological Data by Kernel Maximum Mean Discrepancy.”</span> Journal Article. <em>Bioinformatics</em> 22 (14): e49–57. <a href="https://doi.org/10.1093/bioinformatics/btl242">https://doi.org/10.1093/bioinformatics/btl242</a>.
</div>
<div class="csl-entry">
Bost, P., A. Giladi, Y. Liu, Y. Bendjelal, G. Xu, E. David, R. Blecher-Gonen, et al. 2020. <span>“Host-Viral Infection Maps Reveal Signatures of Severe COVID-19 Patients.”</span> Journal Article. <em>Cell</em> 181 (7): 1475–1488 e12. <a href="https://doi.org/10.1016/j.cell.2020.05.006">https://doi.org/10.1016/j.cell.2020.05.006</a>.
</div>
<div class="csl-entry">
Butler, A., P. Hoffman, P. Smibert, E. Papalexi, and R. Satija. 2018. <span>“Integrating Single-Cell Transcriptomic Data Across Different Conditions, Technologies, and Species.”</span> Journal Article. <em>Nat Biotechnol</em> 36 (5): 411–20. <a href="https://doi.org/10.1038/nbt.4096">https://doi.org/10.1038/nbt.4096</a>.
</div>
<div class="csl-entry">
Buttner, M., Z. Miao, F. A. Wolf, S. A. Teichmann, and F. J. Theis. 2019. <span>“A Test Metric for Assessing Single-Cell RNA-Seq Batch Correction.”</span> Journal Article. <em>Nat Methods</em> 16 (1): 43–49. <a href="https://doi.org/10.1038/s41592-018-0254-1">https://doi.org/10.1038/s41592-018-0254-1</a>.
</div>
<div class="csl-entry">
Chen, G., B. Ning, and T. Shi. 2019. <span>“Single-Cell RNA-Seq Technologies and Related Computational Data Analysis.”</span> Journal Article. <em>Front Genet</em> 10: 317. <a href="https://doi.org/10.3389/fgene.2019.00317">https://doi.org/10.3389/fgene.2019.00317</a>.
</div>
<div class="csl-entry">
Chung, W., H. H. Eum, H. O. Lee, K. M. Lee, H. B. Lee, K. T. Kim, H. S. Ryu, et al. 2017. <span>“Single-Cell RNA-Seq Enables Comprehensive Tumour and Immune Cell Profiling in Primary Breast Cancer.”</span> Journal Article. <em>Nat Commun</em> 8: 15081. <a href="https://doi.org/10.1038/ncomms15081">https://doi.org/10.1038/ncomms15081</a>.
</div>
<div class="csl-entry">
Cover, Thomas M. 1999. <em>Elements of Information Theory</em>. Book. John Wiley &amp; Sons.
</div>
<div class="csl-entry">
Crow, M., A. Paul, S. Ballouz, Z. J. Huang, and J. Gillis. 2018. <span>“Characterizing the Replicability of Cell Types Defined by Single Cell RNA-Sequencing Data Using MetaNeighbor.”</span> Journal Article. <em>Nat Commun</em> 9 (1): 884. <a href="https://doi.org/10.1038/s41467-018-03282-0">https://doi.org/10.1038/s41467-018-03282-0</a>.
</div>
<div class="csl-entry">
Deng, Y., F. Bao, Q. Dai, L. F. Wu, and S. J. Altschuler. 2019. <span>“Scalable Analysis of Cell-Type Composition from Single-Cell Transcriptomics Using Deep Recurrent Learning.”</span> Journal Article. <em>Nat Methods</em> 16 (4): 311–14. <a href="https://doi.org/10.1038/s41592-019-0353-7">https://doi.org/10.1038/s41592-019-0353-7</a>.
</div>
<div class="csl-entry">
Dijk, D. van, R. Sharma, J. Nainys, K. Yim, P. Kathail, A. J. Carr, C. Burdziak, et al. 2018. <span>“Recovering Gene Interactions from Single-Cell Data Using Data Diffusion.”</span> Journal Article. <em>Cell</em> 174 (3): 716–729 e27. <a href="https://doi.org/10.1016/j.cell.2018.05.061">https://doi.org/10.1016/j.cell.2018.05.061</a>.
</div>
<div class="csl-entry">
Ding, J., A. Condon, and S. P. Shah. 2018. <span>“Interpretable Dimensionality Reduction of Single Cell Transcriptome Data with Deep Generative Models.”</span> Journal Article. <em>Nat Commun</em> 9 (1): 2002. <a href="https://doi.org/10.1038/s41467-018-04368-5">https://doi.org/10.1038/s41467-018-04368-5</a>.
</div>
<div class="csl-entry">
Dong, Z., and G. Alterovitz. 2021. <span>“netAE: Semi-Supervised Dimensionality Reduction of Single-Cell RNA Sequencing to Facilitate Cell Labeling.”</span> Journal Article. <em>Bioinformatics</em> 37 (1): 43–49. <a href="https://doi.org/10.1093/bioinformatics/btaa669">https://doi.org/10.1093/bioinformatics/btaa669</a>.
</div>
<div class="csl-entry">
Duvenaud, et al., D. 2015. <span>“Advances in Neural Information Processing Systems 28.”</span> Journal Article, 2224–32.
</div>
<div class="csl-entry">
Eisenstein, M. 2020. <span>“Single-Cell RNA-Seq Analysis Software Providers Scramble to Offer Solutions.”</span> Journal Article. <em>Nat Biotechnol</em> 38 (3): 254–57. <a href="https://doi.org/10.1038/s41587-020-0449-8">https://doi.org/10.1038/s41587-020-0449-8</a>.
</div>
<div class="csl-entry">
Eraslan, G., L. M. Simon, M. Mircea, N. S. Mueller, and F. J. Theis. 2019. <span>“Single-Cell RNA-Seq Denoising Using a Deep Count Autoencoder.”</span> Journal Article. <em>Nat Commun</em> 10 (1): 390. <a href="https://doi.org/10.1038/s41467-018-07931-2">https://doi.org/10.1038/s41467-018-07931-2</a>.
</div>
<div class="csl-entry">
Fabregat, A., S. Jupe, L. Matthews, K. Sidiropoulos, M. Gillespie, P. Garapati, R. Haw, et al. 2018. <span>“The Reactome Pathway Knowledgebase.”</span> Journal Article. <em>Nucleic Acids Res</em> 46 (D1): D649–55. <a href="https://doi.org/10.1093/nar/gkx1132">https://doi.org/10.1093/nar/gkx1132</a>.
</div>
<div class="csl-entry">
Finak, G., A. McDavid, M. Yajima, J. Deng, V. Gersuk, A. K. Shalek, C. K. Slichter, et al. 2015. <span>“MAST: A Flexible Statistical Framework for Assessing Transcriptional Changes and Characterizing Heterogeneity in Single-Cell RNA Sequencing Data.”</span> Journal Article. <em>Genome Biol</em> 16: 278. <a href="https://doi.org/10.1186/s13059-015-0844-5">https://doi.org/10.1186/s13059-015-0844-5</a>.
</div>
<div class="csl-entry">
Fowlkes, E. B., and C. L. Mallows. 1983. <span>“A Method for Comparing Two Hierarchical Clusterings.”</span> Journal Article. <em>Journal of the American Statistical Association</em> 78 (383): 553–69.
</div>
<div class="csl-entry">
Ge, S., H. Wang, A. Alavi, E. Xing, and Z. Bar-Joseph. 2021. <span>“Supervised Adversarial Alignment of Single-Cell RNA-Seq Data.”</span> Journal Article. <em>J Comput Biol</em> 28 (5): 501–13. <a href="https://doi.org/10.1089/cmb.2020.0439">https://doi.org/10.1089/cmb.2020.0439</a>.
</div>
<div class="csl-entry">
Gronbech, C. H., M. F. Vording, P. N. Timshel, C. K. Sonderby, T. H. Pers, and O. Winther. 2020. <span>“scVAE: Variational Auto-Encoders for Single-Cell Gene Expression Data.”</span> Journal Article. <em>Bioinformatics</em> 36 (16): 4415–22. <a href="https://doi.org/10.1093/bioinformatics/btaa293">https://doi.org/10.1093/bioinformatics/btaa293</a>.
</div>
<div class="csl-entry">
Guo, Gao, X., and J. Yin. 2017. <span>“Improved Deep Embedded Clustering with Local Structure Preservation.”</span> Journal Article. <em>Proc. 26th International Joint Conference on Artificial Integlligence</em>, 1753–59.
</div>
<div class="csl-entry">
Haber, A. L., M. Biton, N. Rogel, R. H. Herbst, K. Shekhar, C. Smillie, G. Burgin, et al. 2017. <span>“A Single-Cell Survey of the Small Intestinal Epithelium.”</span> Journal Article. <em>Nature</em> 551 (7680): 333–39. <a href="https://doi.org/10.1038/nature24489">https://doi.org/10.1038/nature24489</a>.
</div>
<div class="csl-entry">
Hadsell, Raia, Sumit Chopra, and Yann LeCun. n.d. <span>“Dimensionality Reduction by Learning an Invariant Mapping.”</span> Conference Proceedings. In <em>2006 IEEE Computer Society Conference on Computer Vision and Pattern Recognition (CVPR’06)</em>, 2:1735–42. IEEE.
</div>
<div class="csl-entry">
Hafemeister, C., and R. Satija. 2019. <span>“Normalization and Variance Stabilization of Single-Cell RNA-Seq Data Using Regularized Negative Binomial Regression.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 296. <a href="https://doi.org/10.1186/s13059-019-1874-1">https://doi.org/10.1186/s13059-019-1874-1</a>.
</div>
<div class="csl-entry">
Hagai, T., X. Chen, R. J. Miragaia, R. Rostom, T. Gomes, N. Kunowska, J. Henriksson, et al. 2018. <span>“Gene Expression Variability Across Cells and Species Shapes Innate Immunity.”</span> Journal Article. <em>Nature</em> 563 (7730): 197–202. <a href="https://doi.org/10.1038/s41586-018-0657-2">https://doi.org/10.1038/s41586-018-0657-2</a>.
</div>
<div class="csl-entry">
Haghverdi, L., A. T. L. Lun, M. D. Morgan, and J. C. Marioni. 2018. <span>“Batch Effects in Single-Cell RNA-Sequencing Data Are Corrected by Matching Mutual Nearest Neighbors.”</span> Journal Article. <em>Nat Biotechnol</em> 36 (5): 421–27. <a href="https://doi.org/10.1038/nbt.4091">https://doi.org/10.1038/nbt.4091</a>.
</div>
<div class="csl-entry">
Hie, B., B. Bryson, and B. Berger. 2019. <span>“Efficient Integration of Heterogeneous Single-Cell Transcriptomes Using Scanorama.”</span> Journal Article. <em>Nat Biotechnol</em> 37 (6): 685–91. <a href="https://doi.org/10.1038/s41587-019-0113-3">https://doi.org/10.1038/s41587-019-0113-3</a>.
</div>
<div class="csl-entry">
Hou, W., Z. Ji, H. Ji, and S. C. Hicks. 2020. <span>“A Systematic Evaluation of Single-Cell RNA-Sequencing Imputation Methods.”</span> Journal Article. <em>Genome Biol</em> 21 (1): 218. <a href="https://doi.org/10.1186/s13059-020-02132-x">https://doi.org/10.1186/s13059-020-02132-x</a>.
</div>
<div class="csl-entry">
Huang, M., J. Wang, E. Torre, H. Dueck, S. Shaffer, R. Bonasio, J. I. Murray, A. Raj, M. Li, and N. R. Zhang. 2018. <span>“SAVER: Gene Expression Recovery for Single-Cell RNA Sequencing.”</span> Journal Article. <em>Nat Methods</em> 15 (7): 539–42. <a href="https://doi.org/10.1038/s41592-018-0033-z">https://doi.org/10.1038/s41592-018-0033-z</a>.
</div>
<div class="csl-entry">
Hubert, L., and P. Arabie. 1985. <span>“Comparing Partitions.”</span> Journal Article. <em>Journal of Classification</em> 2: 193–218.
</div>
<div class="csl-entry">
Huynh-Thu, V. A., A. Irrthum, L. Wehenkel, and P. Geurts. 2010. <span>“Inferring Regulatory Networks from Expression Data Using Tree-Based Methods.”</span> Journal Article. <em>PLoS One</em> 5 (9). <a href="https://doi.org/10.1371/journal.pone.0012776">https://doi.org/10.1371/journal.pone.0012776</a>.
</div>
<div class="csl-entry">
Jang E., Gu S., and Poole B. 2016. <span>“Categorical Reparameterization with Gumbel-Softmax.”</span> Journal Article. <em>arXiv</em>.
</div>
<div class="csl-entry">
Kanehisa, M., M. Furumichi, M. Tanabe, Y. Sato, and K. Morishima. 2017. <span>“KEGG: New Perspectives on Genomes, Pathways, Diseases and Drugs.”</span> Journal Article. <em>Nucleic Acids Res</em> 45 (D1): D353–61. <a href="https://doi.org/10.1093/nar/gkw1092">https://doi.org/10.1093/nar/gkw1092</a>.
</div>
<div class="csl-entry">
Kang, H. M., M. Subramaniam, S. Targ, M. Nguyen, L. Maliskova, E. McCarthy, E. Wan, et al. 2018. <span>“Multiplexed Droplet Single-Cell RNA-Sequencing Using Natural Genetic Variation.”</span> Journal Article. <em>Nat Biotechnol</em> 36 (1): 89–94. <a href="https://doi.org/10.1038/nbt.4042">https://doi.org/10.1038/nbt.4042</a>.
</div>
<div class="csl-entry">
Kharchenko, P. V., L. Silberstein, and D. T. Scadden. 2014. <span>“Bayesian Approach to Single-Cell Differential Expression Analysis.”</span> Journal Article. <em>Nat Methods</em> 11 (7): 740–42. <a href="https://doi.org/10.1038/nmeth.2967">https://doi.org/10.1038/nmeth.2967</a>.
</div>
<div class="csl-entry">
Kinker, G. S., A. C. Greenwald, R. Tal, Z. Orlova, M. S. Cuoco, J. M. McFarland, A. Warren, et al. 2020. <span>“Pan-Cancer Single-Cell RNA-Seq Identifies Recurring Programs of Cellular Heterogeneity.”</span> Journal Article. <em>Nat Genet</em> 52 (11): 1208–18. <a href="https://doi.org/10.1038/s41588-020-00726-6">https://doi.org/10.1038/s41588-020-00726-6</a>.
</div>
<div class="csl-entry">
Korsunsky, I., N. Millard, J. Fan, K. Slowikowski, F. Zhang, K. Wei, Y. Baglaenko, M. Brenner, P. R. Loh, and S. Raychaudhuri. 2019. <span>“Fast, Sensitive and Accurate Integration of Single-Cell Data with Harmony.”</span> Journal Article. <em>Nat Methods</em> 16 (12): 1289–96. <a href="https://doi.org/10.1038/s41592-019-0619-0">https://doi.org/10.1038/s41592-019-0619-0</a>.
</div>
<div class="csl-entry">
Krishnaswamy, S., M. H. Spitzer, M. Mingueneau, S. C. Bendall, O. Litvin, E. Stone, D. Pe’er, and G. P. Nolan. 2014. <span>“Systems Biology. Conditional Density-Based Analysis of t Cell Signaling in Single-Cell Data.”</span> Journal Article. <em>Science</em> 346 (6213): 1250689. <a href="https://doi.org/10.1126/science.1250689">https://doi.org/10.1126/science.1250689</a>.
</div>
<div class="csl-entry">
Lahnemann, D., J. Koster, E. Szczurek, D. J. McCarthy, S. C. Hicks, M. D. Robinson, C. A. Vallejos, et al. 2020. <span>“Eleven Grand Challenges in Single-Cell Data Science.”</span> Journal Article. <em>Genome Biol</em> 21 (1): 31. <a href="https://doi.org/10.1186/s13059-020-1926-6">https://doi.org/10.1186/s13059-020-1926-6</a>.
</div>
<div class="csl-entry">
Levine, J. H., E. F. Simonds, S. C. Bendall, K. L. Davis, A. D. Amir el, M. D. Tadmor, O. Litvin, et al. 2015. <span>“Data-Driven Phenotypic Dissection of AML Reveals Progenitor-Like Cells That Correlate with Prognosis.”</span> Journal Article. <em>Cell</em> 162 (1): 184–97. <a href="https://doi.org/10.1016/j.cell.2015.05.047">https://doi.org/10.1016/j.cell.2015.05.047</a>.
</div>
<div class="csl-entry">
Li, H., E. T. Courtois, D. Sengupta, Y. Tan, K. H. Chen, J. J. L. Goh, S. L. Kong, et al. 2017. <span>“Reference Component Analysis of Single-Cell Transcriptomes Elucidates Cellular Heterogeneity in Human Colorectal Tumors.”</span> Journal Article. <em>Nat Genet</em> 49 (5): 708–18. <a href="https://doi.org/10.1038/ng.3818">https://doi.org/10.1038/ng.3818</a>.
</div>
<div class="csl-entry">
Li, W. V., and J. J. Li. 2018. <span>“An Accurate and Robust Imputation Method scImpute for Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Nat Commun</em> 9 (1): 997. <a href="https://doi.org/10.1038/s41467-018-03405-7">https://doi.org/10.1038/s41467-018-03405-7</a>.
</div>
<div class="csl-entry">
Lieberman, Y., L. Rokach, and T. Shay. 2018. <span>“CaSTLe - Classification of Single Cells by Transfer Learning: Harnessing the Power of Publicly Available Single Cell RNA Sequencing Experiments to Annotate New Experiments.”</span> Journal Article. <em>PLoS One</em> 13 (10): e0205499. <a href="https://doi.org/10.1371/journal.pone.0205499">https://doi.org/10.1371/journal.pone.0205499</a>.
</div>
<div class="csl-entry">
Lin, C., S. Jain, H. Kim, and Z. Bar-Joseph. 2017. <span>“Using Neural Networks for Reducing the Dimensions of Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Nucleic Acids Res</em> 45 (17): e156. <a href="https://doi.org/10.1093/nar/gkx681">https://doi.org/10.1093/nar/gkx681</a>.
</div>
<div class="csl-entry">
Lin, P., M. Troup, and J. W. Ho. 2017. <span>“CIDR: Ultrafast and Accurate Clustering Through Imputation for Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Genome Biol</em> 18 (1): 59. <a href="https://doi.org/10.1186/s13059-017-1188-0">https://doi.org/10.1186/s13059-017-1188-0</a>.
</div>
<div class="csl-entry">
Lindenbaum, Stanley, O., and S. Krishnaswamy. 2018. <span>“Geometry-Based Data Generation.”</span> Journal Article. <em>Advances in Neural Information Processing Systems</em>.
</div>
<div class="csl-entry">
Linderman, G. C., M. Rachh, J. G. Hoskins, S. Steinerberger, and Y. Kluger. 2019. <span>“Fast Interpolation-Based t-SNE for Improved Visualization of Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Nat Methods</em> 16 (3): 243–45. <a href="https://doi.org/10.1038/s41592-018-0308-4">https://doi.org/10.1038/s41592-018-0308-4</a>.
</div>
<div class="csl-entry">
Lopez, R., J. Regier, M. B. Cole, M. I. Jordan, and N. Yosef. 2018. <span>“Deep Generative Modeling for Single-Cell Transcriptomics.”</span> Journal Article. <em>Nat Methods</em> 15 (12): 1053–58. <a href="https://doi.org/10.1038/s41592-018-0229-2">https://doi.org/10.1038/s41592-018-0229-2</a>.
</div>
<div class="csl-entry">
Lotfollahi, M., F. A. Wolf, and F. J. Theis. 2019. <span>“scGen Predicts Single-Cell Perturbation Responses.”</span> Journal Article. <em>Nat Methods</em> 16 (8): 715–21. <a href="https://doi.org/10.1038/s41592-019-0494-8">https://doi.org/10.1038/s41592-019-0494-8</a>.
</div>
<div class="csl-entry">
Lun, A. T., K. Bach, and J. C. Marioni. 2016. <span>“Pooling Across Cells to Normalize Single-Cell RNA Sequencing Data with Many Zero Counts.”</span> Journal Article. <em>Genome Biol</em> 17: 75. <a href="https://doi.org/10.1186/s13059-016-0947-7">https://doi.org/10.1186/s13059-016-0947-7</a>.
</div>
<div class="csl-entry">
Maaten, G. van der, L. &amp; Hinton. 2008. <span>“Visualizing Data Using t-SNE.”</span> Journal Article. <em>J. Mach. Learn</em> 9: 2579–2605.
</div>
<div class="csl-entry">
Macosko, E. Z., A. Basu, R. Satija, J. Nemesh, K. Shekhar, M. Goldman, I. Tirosh, et al. 2015. <span>“Highly Parallel Genome-Wide Expression Profiling of Individual Cells Using Nanoliter Droplets.”</span> Journal Article. <em>Cell</em> 161 (5): 1202–14. <a href="https://doi.org/10.1016/j.cell.2015.05.002">https://doi.org/10.1016/j.cell.2015.05.002</a>.
</div>
<div class="csl-entry">
Mannarapu, M., B. Dariya, and O. R. Bandapalli. 2021. <span>“Application of Single-Cell Sequencing Technologies in Pancreatic Cancer.”</span> Journal Article. <em>Mol Cell Biochem</em> 476 (6): 2429–37. <a href="https://doi.org/10.1007/s11010-021-04095-4">https://doi.org/10.1007/s11010-021-04095-4</a>.
</div>
<div class="csl-entry">
Marouf, M., P. Machart, V. Bansal, C. Kilian, D. S. Magruder, C. F. Krebs, and S. Bonn. 2020. <span>“Realistic in Silico Generation and Augmentation of Single-Cell RNA-Seq Data Using Generative Adversarial Networks.”</span> Journal Article. <em>Nat Commun</em> 11 (1): 166. <a href="https://doi.org/10.1038/s41467-019-14018-z">https://doi.org/10.1038/s41467-019-14018-z</a>.
</div>
<div class="csl-entry">
McGinnis, C. S., L. M. Murrow, and Z. J. Gartner. 2019. <span>“DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors.”</span> Journal Article. <em>Cell Syst</em> 8 (4): 329–337 e4. <a href="https://doi.org/10.1016/j.cels.2019.03.003">https://doi.org/10.1016/j.cels.2019.03.003</a>.
</div>
<div class="csl-entry">
McInnes, Healy, L. 2018. <span>“Umap: Uniform Manifold Approximation and Projection for Dimension Reduction.”</span> Journal Article. <em>ArXiv</em>. https://doi.org/<a href=" https://arxiv.org/
abs/1802.03426"> https://arxiv.org/
abs/1802.03426</a>.
</div>
<div class="csl-entry">
Miao, Z., K. Deng, X. Wang, and X. Zhang. 2018. <span>“DEsingle for Detecting Three Types of Differential Expression in Single-Cell RNA-Seq Data.”</span> Journal Article. <em>Bioinformatics</em> 34 (18): 3223–24. <a href="https://doi.org/10.1093/bioinformatics/bty332">https://doi.org/10.1093/bioinformatics/bty332</a>.
</div>
<div class="csl-entry">
Miyato, T., and M Koyama. 2018. <span>“cGANs with Projection Discriminator.”</span> Journal Article. <em>Preprint</em>. https://doi.org/<a href="https://arxiv.org/abs/1802.05637 ">https://arxiv.org/abs/1802.05637</a>.
</div>
<div class="csl-entry">
Moon, K. R. et al. 2017. <span>“PHATE: A Dimensionality Reduction Method for Visualizing Trajectory Structures in High-Dimensional Biological Data.”</span> Journal Article. <em>bioRxiv</em>. https://doi.org/<a href="https://doi.org/10.1101/120378">https://doi.org/10.1101/120378</a>.
</div>
<div class="csl-entry">
Navin, N. E. 2015. <span>“The First Five Years of Single-Cell Cancer Genomics and Beyond.”</span> Journal Article. <em>Genome Res</em> 25 (10): 1499–1507. <a href="https://doi.org/10.1101/gr.191098.115">https://doi.org/10.1101/gr.191098.115</a>.
</div>
<div class="csl-entry">
Newman, M. E. 2006. <span>“Modularity and Community Structure in Networks.”</span> Journal Article. <em>Proc Natl Acad Sci U S A</em> 103 (23): 8577–82. <a href="https://doi.org/10.1073/pnas.0601602103">https://doi.org/10.1073/pnas.0601602103</a>.
</div>
<div class="csl-entry">
Patel, N. D., S. K. Nguang, and G. G. Coghill. 2007. <span>“Neural Network Implementation Using Bit Streams.”</span> Journal Article. <em>IEEE Trans Neural Netw</em> 18 (5): 1488–1504. <a href="https://doi.org/10.1109/tnn.2007.895822">https://doi.org/10.1109/tnn.2007.895822</a>.
</div>
<div class="csl-entry">
Peng, T., Q. Zhu, P. Yin, and K. Tan. 2019. <span>“SCRABBLE: Single-Cell RNA-Seq Imputation Constrained by Bulk RNA-Seq Data.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 88. <a href="https://doi.org/10.1186/s13059-019-1681-8">https://doi.org/10.1186/s13059-019-1681-8</a>.
</div>
<div class="csl-entry">
Petegrosso, R., Z. Li, and R. Kuang. 2020. <span>“Machine Learning and Statistical Methods for Clustering Single-Cell RNA-Sequencing Data.”</span> Journal Article. <em>Brief Bioinform</em> 21 (4): 1209–23. <a href="https://doi.org/10.1093/bib/bbz063">https://doi.org/10.1093/bib/bbz063</a>.
</div>
<div class="csl-entry">
Picelli, S., A. K. Bjorklund, O. R. Faridani, S. Sagasser, G. Winberg, and R. Sandberg. 2013. <span>“Smart-Seq2 for Sensitive Full-Length Transcriptome Profiling in Single Cells.”</span> Journal Article. <em>Nat Methods</em> 10 (11): 1096–98. <a href="https://doi.org/10.1038/nmeth.2639">https://doi.org/10.1038/nmeth.2639</a>.
</div>
<div class="csl-entry">
Polanski, K., M. D. Young, Z. Miao, K. B. Meyer, S. A. Teichmann, and J. E. Park. 2020. <span>“BBKNN: Fast Batch Alignment of Single Cell Transcriptomes.”</span> Journal Article. <em>Bioinformatics</em> 36 (3): 964–65. <a href="https://doi.org/10.1093/bioinformatics/btz625">https://doi.org/10.1093/bioinformatics/btz625</a>.
</div>
<div class="csl-entry">
Qiu, X., Q. Mao, Y. Tang, L. Wang, R. Chawla, H. A. Pliner, and C. Trapnell. 2017. <span>“Reversed Graph Embedding Resolves Complex Single-Cell Trajectories.”</span> Journal Article. <em>Nat Methods</em> 14 (10): 979–82. <a href="https://doi.org/10.1038/nmeth.4402">https://doi.org/10.1038/nmeth.4402</a>.
</div>
<div class="csl-entry">
Racle, J., K. de Jonge, P. Baumgaertner, D. E. Speiser, and D. Gfeller. 2017. <span>“Simultaneous Enumeration of Cancer and Immune Cell Types from Bulk Tumor Gene Expression Data.”</span> Journal Article. <em>Elife</em> 6. <a href="https://doi.org/10.7554/eLife.26476">https://doi.org/10.7554/eLife.26476</a>.
</div>
<div class="csl-entry">
Rashid, S., S. Shah, Z. Bar-Joseph, and R. Pandya. 2019. <span>“Dhaka: Variational Autoencoder for Unmasking Tumor Heterogeneity from Single Cell Genomic Data.”</span> Journal Article. <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btz095">https://doi.org/10.1093/bioinformatics/btz095</a>.
</div>
<div class="csl-entry">
Regev, A., S. A. Teichmann, E. S. Lander, I. Amit, C. Benoist, E. Birney, B. Bodenmiller, et al. 2017. <span>“The Human Cell Atlas.”</span> Journal Article. <em>Elife</em> 6. <a href="https://doi.org/10.7554/eLife.27041">https://doi.org/10.7554/eLife.27041</a>.
</div>
<div class="csl-entry">
Rosenberg, A., and J. Hirschberg. 2007. <span>“Proceedings of the 2007 Joint Conference on Empirical Methods in Natural Language Processing and Computational Natural Language Learning (EMNLP-CoNLL).”</span> Journal Article, 410–20.
</div>
<div class="csl-entry">
Rousseeuw, P. J. 1987. <span>“Silhouettes: A Graphical Aid to the Interpretation and Validation of Cluster Analysis.”</span> Journal Article. <em>Journal of Computational and Applied Mathematics</em> 20: 583–617.
</div>
<div class="csl-entry">
Roweis, S. T., and L. K. Saul. 2000. <span>“Nonlinear Dimensionality Reduction by Locally Linear Embedding.”</span> Journal Article. <em>Science</em> 290 (5500): 2323–26. <a href="https://doi.org/10.1126/science.290.5500.2323">https://doi.org/10.1126/science.290.5500.2323</a>.
</div>
<div class="csl-entry">
Shekhar, K., S. W. Lapan, I. E. Whitney, N. M. Tran, E. Z. Macosko, M. Kowalczyk, X. Adiconis, et al. 2016. <span>“Comprehensive Classification of Retinal Bipolar Neurons by Single-Cell Transcriptomics.”</span> Journal Article. <em>Cell</em> 166 (5): 1308–1323 e30. <a href="https://doi.org/10.1016/j.cell.2016.07.054">https://doi.org/10.1016/j.cell.2016.07.054</a>.
</div>
<div class="csl-entry">
Srinivasan, S., A. Leshchyk, N. T. Johnson, and D. Korkin. 2020. <span>“A Hybrid Deep Clustering Approach for Robust Cell Type Profiling Using Single-Cell RNA-Seq Data.”</span> Journal Article. <em>RNA</em> 26 (10): 1303–19. <a href="https://doi.org/10.1261/rna.074427.119">https://doi.org/10.1261/rna.074427.119</a>.
</div>
<div class="csl-entry">
Strehland, A., and J. Ghosh. 2002. <span>“Cluster Ensembles—a Knowledge Reuse Framework for Combining Multiple Partitions.”</span> Journal Article. <em>J Mach Learn Res</em> 3: 583–617.
</div>
<div class="csl-entry">
Stuart, T., A. Butler, P. Hoffman, C. Hafemeister, E. Papalexi, 3rd Mauck W. M., Y. Hao, M. Stoeckius, P. Smibert, and R. Satija. 2019. <span>“Comprehensive Integration of Single-Cell Data.”</span> Journal Article. <em>Cell</em> 177 (7): 1888–1902 e21. <a href="https://doi.org/10.1016/j.cell.2019.05.031">https://doi.org/10.1016/j.cell.2019.05.031</a>.
</div>
<div class="csl-entry">
Subelj, L., and M. Bajec. 2011. <span>“Unfolding Communities in Large Complex Networks: Combining Defensive and Offensive Label Propagation for Core Extraction.”</span> Journal Article. <em>Phys Rev E Stat Nonlin Soft Matter Phys</em> 83 (3 Pt 2): 036103. <a href="https://doi.org/10.1103/PhysRevE.83.036103">https://doi.org/10.1103/PhysRevE.83.036103</a>.
</div>
<div class="csl-entry">
Subramanian, A., P. Tamayo, V. K. Mootha, S. Mukherjee, B. L. Ebert, M. A. Gillette, A. Paulovich, et al. 2005. <span>“Gene Set Enrichment Analysis: A Knowledge-Based Approach for Interpreting Genome-Wide Expression Profiles.”</span> Journal Article. <em>Proc Natl Acad Sci U S A</em> 102 (43): 15545–50. <a href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a>.
</div>
<div class="csl-entry">
Sun, Zhu, S., and X. Zhou. 2019. <span>“Accuracy, Robustness and Scalability of Dimensionality Reduction Methods for Single-Cell RNA-Seq Analysis.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 269. <a href="https://doi.org/10.1186/s13059-019-1898-6">https://doi.org/10.1186/s13059-019-1898-6</a>.
</div>
<div class="csl-entry">
Svensson, V., A. Gayoso, N. Yosef, and L. Pachter. 2020. <span>“Interpretable Factor Models of Single-Cell RNA-Seq via Variational Autoencoders.”</span> Journal Article. <em>Bioinformatics</em> 36 (11): 3418–21. <a href="https://doi.org/10.1093/bioinformatics/btaa169">https://doi.org/10.1093/bioinformatics/btaa169</a>.
</div>
<div class="csl-entry">
Tian, Wan, T. 2019. <span>“Clustering Single-Cell RNA-Seq Data with a Model-Based Deep Learning Approach.”</span> Journal Article. <em>Nat Mach Intell</em> 1. https://doi.org/<a href=" https://doi.org/10.1038/s42256-019-0037-0"> https://doi.org/10.1038/s42256-019-0037-0</a>.
</div>
<div class="csl-entry">
Tirosh, I., B. Izar, S. M. Prakadan, 2nd Wadsworth M. H., D. Treacy, J. J. Trombetta, A. Rotem, et al. 2016. <span>“Dissecting the Multicellular Ecosystem of Metastatic Melanoma by Single-Cell RNA-Seq.”</span> Journal Article. <em>Science</em> 352 (6282): 189–96. <a href="https://doi.org/10.1126/science.aad0501">https://doi.org/10.1126/science.aad0501</a>.
</div>
<div class="csl-entry">
Torroja, C., and F. Sanchez-Cabo. 2019. <span>“Digitaldlsorter: Deep-Learning on scRNA-Seq to Deconvolute Gene Expression Data.”</span> Journal Article. <em>Front Genet</em> 10: 978. <a href="https://doi.org/10.3389/fgene.2019.00978">https://doi.org/10.3389/fgene.2019.00978</a>.
</div>
<div class="csl-entry">
Traag, V. A., L. Waltman, and N. J. van Eck. 2019. <span>“From Louvain to Leiden: Guaranteeing Well-Connected Communities.”</span> Journal Article. <em>Sci Rep</em> 9 (1): 5233. <a href="https://doi.org/10.1038/s41598-019-41695-z">https://doi.org/10.1038/s41598-019-41695-z</a>.
</div>
<div class="csl-entry">
Tran, H. T. N., K. S. Ang, M. Chevrier, X. Zhang, N. Y. S. Lee, M. Goh, and J. Chen. 2020. <span>“A Benchmark of Batch-Effect Correction Methods for Single-Cell RNA Sequencing Data.”</span> Journal Article. <em>Genome Biol</em> 21 (1): 12. <a href="https://doi.org/10.1186/s13059-019-1850-9">https://doi.org/10.1186/s13059-019-1850-9</a>.
</div>
<div class="csl-entry">
Vallejos, C. A., J. C. Marioni, and S. Richardson. 2015. <span>“BASiCS: Bayesian Analysis of Single-Cell Sequencing Data.”</span> Journal Article. <em>PLoS Comput Biol</em> 11 (6): e1004333. <a href="https://doi.org/10.1371/journal.pcbi.1004333">https://doi.org/10.1371/journal.pcbi.1004333</a>.
</div>
<div class="csl-entry">
Vitak, S. A., K. A. Torkenczy, J. L. Rosenkrantz, A. J. Fields, L. Christiansen, M. H. Wong, L. Carbone, F. J. Steemers, and A. Adey. 2017. <span>“Sequencing Thousands of Single-Cell Genomes with Combinatorial Indexing.”</span> Journal Article. <em>Nat Methods</em> 14 (3): 302–8. <a href="https://doi.org/10.1038/nmeth.4154">https://doi.org/10.1038/nmeth.4154</a>.
</div>
<div class="csl-entry">
Wang, B., J. Zhu, E. Pierson, D. Ramazzotti, and S. Batzoglou. 2017. <span>“Visualization and Analysis of Single-Cell RNA-Seq Data by Kernel-Based Similarity Learning.”</span> Journal Article. <em>Nat Methods</em> 14 (4): 414–16. <a href="https://doi.org/10.1038/nmeth.4207">https://doi.org/10.1038/nmeth.4207</a>.
</div>
<div class="csl-entry">
Wang, D., and J. Gu. 2018. <span>“VASC: Dimension Reduction and Visualization of Single-Cell RNA-Seq Data by Deep Variational Autoencoder.”</span> Journal Article. <em>Genomics Proteomics Bioinformatics</em> 16 (5): 320–31. <a href="https://doi.org/10.1016/j.gpb.2018.08.003">https://doi.org/10.1016/j.gpb.2018.08.003</a>.
</div>
<div class="csl-entry">
Wang, D., S. Hou, L. Zhang, X. Wang, B. Liu, and Z. Zhang. 2021. <span>“iMAP: Integration of Multiple Single-Cell Datasets by Adversarial Paired Transfer Networks.”</span> Journal Article. <em>Genome Biol</em> 22 (1): 63. <a href="https://doi.org/10.1186/s13059-021-02280-8">https://doi.org/10.1186/s13059-021-02280-8</a>.
</div>
<div class="csl-entry">
Wang, J., D. Agarwal, M. Huang, G. Hu, Z. Zhou, C. Ye, and N. R. Zhang. 2019. <span>“Data Denoising with Transfer Learning in Single-Cell Transcriptomics.”</span> Journal Article. <em>Nat Methods</em> 16 (9): 875–78. <a href="https://doi.org/10.1038/s41592-019-0537-1">https://doi.org/10.1038/s41592-019-0537-1</a>.
</div>
<div class="csl-entry">
Wang, Nie, L. 2020. <span>“An Interpretable Deep-Learning Architecture of Capsule Networks for Identifying Cell-Type Gene Expression Programs from Single-Cell RNA-Sequencing Data.”</span> Journal Article. <em>Nat Mach Intell</em> 2: 693–703. https://doi.org/<a href="ttps://doi.org/10.1038/s42256-020-00244-4">ttps://doi.org/10.1038/s42256-020-00244-4</a>.
</div>
<div class="csl-entry">
Wang, T., T. S. Johnson, W. Shao, Z. Lu, B. R. Helm, J. Zhang, and K. Huang. 2019. <span>“BERMUDA: A Novel Deep Transfer Learning Method for Single-Cell RNA Sequencing Batch Correction Reveals Hidden High-Resolution Cellular Subtypes.”</span> Journal Article. <em>Genome Biol</em> 20 (1): 165. <a href="https://doi.org/10.1186/s13059-019-1764-6">https://doi.org/10.1186/s13059-019-1764-6</a>.
</div>
<div class="csl-entry">
Wang, Y. X., M. S. Waterman, and H. Huang. 2014. <span>“Gene Coexpression Measures in Large Heterogeneous Samples Using Count Statistics.”</span> Journal Article. <em>Proc Natl Acad Sci U S A</em> 111 (46): 16371–76. <a href="https://doi.org/10.1073/pnas.1417128111">https://doi.org/10.1073/pnas.1417128111</a>.
</div>
<div class="csl-entry">
Wauters, E., P. Van Mol, A. D. Garg, S. Jansen, Y. Van Herck, L. Vanderbeke, A. Bassez, et al. 2021. <span>“Discriminating Mild from Critical COVID-19 by Innate and Adaptive Immune Single-Cell Profiling of Bronchoalveolar Lavages.”</span> Journal Article. <em>Cell Res</em> 31 (3): 272–90. <a href="https://doi.org/10.1038/s41422-020-00455-9">https://doi.org/10.1038/s41422-020-00455-9</a>.
</div>
<div class="csl-entry">
Welch, J. D., A. J. Hartemink, and J. F. Prins. 2016. <span>“SLICER: Inferring Branched, Nonlinear Cellular Trajectories from Single Cell RNA-Seq Data.”</span> Journal Article. <em>Genome Biol</em> 17 (1): 106. <a href="https://doi.org/10.1186/s13059-016-0975-3">https://doi.org/10.1186/s13059-016-0975-3</a>.
</div>
<div class="csl-entry">
Wolf, F. A., P. Angerer, and F. J. Theis. 2018. <span>“SCANPY: Large-Scale Single-Cell Gene Expression Data Analysis.”</span> Journal Article. <em>Genome Biol</em> 19 (1): 15. <a href="https://doi.org/10.1186/s13059-017-1382-0">https://doi.org/10.1186/s13059-017-1382-0</a>.
</div>
<div class="csl-entry">
Wolock, S. L., R. Lopez, and A. M. Klein. 2019. <span>“Scrublet: Computational Identification of Cell Doublets in Single-Cell Transcriptomic Data.”</span> Journal Article. <em>Cell Syst</em> 8 (4): 281–291 e9. <a href="https://doi.org/10.1016/j.cels.2018.11.005">https://doi.org/10.1016/j.cels.2018.11.005</a>.
</div>
<div class="csl-entry">
Xie, Junyuan, Ross Girshick, and Ali Farhadi. n.d. <span>“Unsupervised Deep Embedding for Clustering Analysis.”</span> Conference Proceedings. In <em>International Conference on Machine Learning</em>, 478–87. PMLR.
</div>
<div class="csl-entry">
Xu, C., R. Lopez, E. Mehlman, J. Regier, M. I. Jordan, and N. Yosef. 2021. <span>“Probabilistic Harmonization and Annotation of Single-Cell Transcriptomics Data with Deep Generative Models.”</span> Journal Article. <em>Mol Syst Biol</em> 17 (1): e9620. <a href="https://doi.org/10.15252/msb.20209620">https://doi.org/10.15252/msb.20209620</a>.
</div>
<div class="csl-entry">
Xu, Y., Z. Zhang, L. You, J. Liu, Z. Fan, and X. Zhou. 2020. <span>“scIGANs: Single-Cell RNA-Seq Imputation Using Generative Adversarial Networks.”</span> Journal Article. <em>Nucleic Acids Res</em> 48 (15): e85. <a href="https://doi.org/10.1093/nar/gkaa506">https://doi.org/10.1093/nar/gkaa506</a>.
</div>
<div class="csl-entry">
Yevshin, I., R. Sharipov, T. Valeev, A. Kel, and F. Kolpakov. 2017. <span>“GTRD: A Database of Transcription Factor Binding Sites Identified by ChIP-Seq Experiments.”</span> Journal Article. <em>Nucleic Acids Res</em> 45 (D1): D61–67. <a href="https://doi.org/10.1093/nar/gkw951">https://doi.org/10.1093/nar/gkw951</a>.
</div>
<div class="csl-entry">
Yoshihara, K., M. Shahmoradgoli, E. Martinez, R. Vegesna, H. Kim, W. Torres-Garcia, V. Trevino, et al. 2013. <span>“Inferring Tumour Purity and Stromal and Immune Cell Admixture from Expression Data.”</span> Journal Article. <em>Nat Commun</em> 4: 2612. <a href="https://doi.org/10.1038/ncomms3612">https://doi.org/10.1038/ncomms3612</a>.
</div>
<div class="csl-entry">
Yu, B., C. Chen, R. Qi, R. Zheng, P. J. Skillman-Lawrence, X. Wang, A. Ma, and H. Gu. 2020. <span>“scGMAI: A Gaussian Mixture Model for Clustering Single-Cell RNA-Seq Data Based on Deep Autoencoder.”</span> Journal Article. <em>Brief Bioinform</em>. <a href="https://doi.org/10.1093/bib/bbaa316">https://doi.org/10.1093/bib/bbaa316</a>.
</div>
<div class="csl-entry">
Yuan, Y., and Z. Bar-Joseph. 2019. <span>“Deep Learning for Inferring Gene Relationships from Single-Cell Expression Data.”</span> Journal Article. <em>Proc Natl Acad Sci U S A</em>. <a href="https://doi.org/10.1073/pnas.1911536116">https://doi.org/10.1073/pnas.1911536116</a>.
</div>
<div class="csl-entry">
Zahn, H., A. Steif, E. Laks, P. Eirew, M. VanInsberghe, S. P. Shah, S. Aparicio, and C. L. Hansen. 2017. <span>“Scalable Whole-Genome Single-Cell Library Preparation Without Preamplification.”</span> Journal Article. <em>Nat Methods</em> 14 (2): 167–73. <a href="https://doi.org/10.1038/nmeth.4140">https://doi.org/10.1038/nmeth.4140</a>.
</div>
<div class="csl-entry">
Zappia, L., B. Phipson, and A. Oshlack. 2017. <span>“Splatter: Simulation of Single-Cell RNA Sequencing Data.”</span> Journal Article. <em>Genome Biol</em> 18 (1): 174. <a href="https://doi.org/10.1186/s13059-017-1305-0">https://doi.org/10.1186/s13059-017-1305-0</a>.
</div>
<div class="csl-entry">
Zheng, G. X., J. M. Terry, P. Belgrader, P. Ryvkin, Z. W. Bent, R. Wilson, S. B. Ziraldo, et al. 2017. <span>“Massively Parallel Digital Transcriptional Profiling of Single Cells.”</span> Journal Article. <em>Nat Commun</em> 8: 14049. <a href="https://doi.org/10.1038/ncomms14049">https://doi.org/10.1038/ncomms14049</a>.
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN100" class="csl-entry">
Wolf, F. A., P. Angerer, and F. J. Theis. 2018. <span>“SCANPY: Large-Scale Single-Cell Gene Expression Data Analysis.”</span> Journal Article. <em>Genome Biol</em> 19 (1): 15. <a href="https://doi.org/10.1186/s13059-017-1382-0">https://doi.org/10.1186/s13059-017-1382-0</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="tables.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/Huang-AI4Medicine-Lab/survey-of-DL-for-scRNA-seq-analysis/edit/main/08-Tables.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["survey-of-DL-for-scRNA-seq-analysis.pdf", "survey-of-DL-for-scRNA-seq-analysis.epub"],
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
